From 8a04132c657f6993168b94e52eef953524becf52 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 24 Oct 2013 20:22:00 +0000
Subject: [PATCH 0488/1077] Mantis 0029552: device documentation Mantis
 0028823: All Components: intranet file path changed: adjust comments fixed
 module doxygen documentation

git-svn-id: svn://localhost/SelfServiceCommon/trunk@603 90b65887-3827-0410-9a23-83215b262276
---
 .../inc/resource/appVersionInfoResource.h     |  42 +--
 .../Massai/cpp/AEAParser/src/local_def.cpp    |  29 +-
 .../cpp/BillingService/inc/BillingC_def.h     |  24 +-
 .../cpp/BillingService/inc/BillingS_def.h     |  57 ++--
 .../Massai/cpp/Controller/inc/mstarter_def.h  | 321 +++++++++---------
 .../inc/local_def_gracefullycrash.h           |  30 +-
 .../Massai/cpp/Graphic/src/local_def.cpp      |  40 ++-
 .../cpp/MassaiCorba/inc/localclnt_def.h       |  23 +-
 .../Massai/cpp/MassaiCorba/inc/localsrv_def.h |  23 +-
 .../cpp/MassaiCorba/inc/massaiCORBA_def.h     | 261 +++++++-------
 .../cpp/MassaiCorba_VC6/inc/localclnt_def.h   |  23 +-
 .../cpp/MassaiCorba_VC6/inc/localsrv_def.h    |  23 +-
 .../cpp/MassaiCorba_VC6/inc/massaiCORBA_def.h | 208 ++++++------
 .../inc/StatisticsClnt_def.h                  |  23 +-
 .../StatisticsService/inc/Statistics_def.h    | 127 +++----
 .../Massai/cpp/Systools/src/local_def.cpp     | 144 ++++----
 .../Massai/cpp/Text/src/local_def.cpp         |  50 +--
 .../cpp/Tools/inc/local_def_msystools.h       | 171 +++++-----
 .../Massai/cpp/Trace/inc/TraceDLL_def.h       |  18 +-
 .../Massai/cpp/Trace/inc/TraceHDLDLL_def.h    |  19 +-
 .../Massai/cpp/Trace/inc/local_def.h          | 176 +++++-----
 .../Massai/cpp/Trace/res/TraceDLL.rc          |   2 -
 .../Massai/cpp/initIDL/src/local_def.cpp      |  37 +-
 .../Massai/cpp/textIDL/src/local_def.cpp      |  88 ++---
 .../cpp/TrcServ/inc/NonCryptingTrcServ_def.h  |   7 +-
 .../Services/cpp/TrcServ/inc/decrypt_def.h    |  47 ++-
 26 files changed, 1036 insertions(+), 977 deletions(-)

diff --git a/SelfServiceCommon/Interfaces/inc/resource/appVersionInfoResource.h b/SelfServiceCommon/Interfaces/inc/resource/appVersionInfoResource.h
index e5ffda4b..39e37e83 100644
--- a/SelfServiceCommon/Interfaces/inc/resource/appVersionInfoResource.h
+++ b/SelfServiceCommon/Interfaces/inc/resource/appVersionInfoResource.h
@@ -1,42 +1,8 @@
 #include "Global_def.h"
 
-#include "winver.h"
+#include <windows.h>
 
-#define LANG_US_ENGLISH    0x0409
-#define LANG_US_ENGLISH_T   "0409"
+#define GERMAN_LANG     0x0407
+#define UNICODE_CHARSET 0x04B0
 
-#define CHARSET_WINDOWS    0x04E4
-#define CHARSET_WINDOWS_T   "04E4"
-
-VS_VERSION_INFO VERSIONINFO
-  FILEVERSION    FILE_VER_BIN
-  PRODUCTVERSION PRODUCT_VER_BIN
-  FILEFLAGSMASK  VS_FFI_FILEFLAGSMASK
-#ifdef _DEBUG
-  FILEFLAGS      VS_FF_DEBUG
-#else
-  FILEFLAGS      0x0L
-#endif
-  FILEOS         (VOS_NT | VOS__WINDOWS32)
-  FILETYPE       VFT_APP
-  BEGIN
-    BLOCK "StringFileInfo"
-    BEGIN
-      BLOCK LANG_US_ENGLISH_T CHARSET_WINDOWS_T
-      BEGIN
-        VALUE COMPANY
-        VALUE FILEDESCRIPTION
-        VALUE FILE_VER_STR
-        VALUE INTERNALNAME
-        VALUE COPYRIGHT
-        VALUE LEGALTRADEMARKS
-        VALUE ORIGINALFILENAME
-        VALUE PRODUCT
-        VALUE PRODUCT_VER_STR
-      END
-    END
-    BLOCK "VarFileInfo"
-    BEGIN
-      VALUE "Translation", LANG_US_ENGLISH, CHARSET_WINDOWS
-    END
-  END
+#define GERMAN_UNICODE  "040704B0"
diff --git a/SelfServiceCommon/Massai/cpp/AEAParser/src/local_def.cpp b/SelfServiceCommon/Massai/cpp/AEAParser/src/local_def.cpp
index e6223f5e..5cb0edf4 100644
--- a/SelfServiceCommon/Massai/cpp/AEAParser/src/local_def.cpp
+++ b/SelfServiceCommon/Massai/cpp/AEAParser/src/local_def.cpp
@@ -3,26 +3,31 @@
 VERSION_ID_DT(aea_parser, 1, 0, 0, 4);
 
 /*!
-\page aea_parser aea_parser - Release Notes
+\page aeaParser.lib Release Notes aeaParser.lib
 
-\date 18.07.2013
-\li V1.0.0.4
+@version 1.0.0.4
+
+\section changehistory Change history
+
+aeaParser.lib
+
+\version 1.0.0.4
+\li 18.07.2013
 \li Mantis 0028134: AEASimulation: invalid error messages
 
-\date 26.06.2013
-\li V1.0.0.3
+\version 1.0.0.3
+\li 26.06.2013
 \li Mantis 0027792: AEASimulation: return error for invalid id in clear command
 
-\date 26.06.2013
-\li V1.0.0.2
+\version 1.0.0.2
+\li 26.06.2013
 \li Mantis 0027739: AEASimulation: use load order for id enumeration
 
-\date 19.10.2010
-\li V1.0.0.1
+\version 1.0.0.1
+\li 19.10.2010
 \li JK Moved to SelfServiceCommon, changing #include lines
 
-\date 19.10.2010
-\li V1.0.0.0
+\version 1.0.0.0
+\li 19.10.2010
 \li SteT Initial Version
 */
-
diff --git a/SelfServiceCommon/Massai/cpp/BillingService/inc/BillingC_def.h b/SelfServiceCommon/Massai/cpp/BillingService/inc/BillingC_def.h
index 9edf5318..f94b615e 100644
--- a/SelfServiceCommon/Massai/cpp/BillingService/inc/BillingC_def.h
+++ b/SelfServiceCommon/Massai/cpp/BillingService/inc/BillingC_def.h
@@ -1,9 +1,23 @@
 #define FILE_VER_BIN     1,0,0,0
-#define FILE_VER_STR     "FileVersion", "1.0.0.0\0"
+#define FILE_VER_STR     "FileVersion", "1.0.0.0"
 
-#define FILEDESCRIPTION  "FileDescription"  ,"Billing Client\0"
-#define INTERNALNAME     "InternalName"     ,"Billing\0"
-#define LEGALTRADEMARKS  "LegalTrademarks"  ,"\0"
-#define ORIGINALFILENAME "OriginalFilename" ,"Billingclnt.exe\0"
+/*!
+\page Billingclnt.exe Release Notes - Billingclnt.exe - test client for billing.exe
+
+@version 1.0.0.0
+
+\section changehistory Change history
+
+Billing.exe
+
+\version 1.0.0.0
+\li Initial Version
+*/
+
+
+#define FILEDESCRIPTION  "FileDescription",  "Billing Client"
+#define INTERNALNAME     "InternalName",     "Billingclnt.exe"
+#define ORIGINALFILENAME "OriginalFilename", "Billingclnt.exe"
+#define LEGALTRADEMARKS  "LegalTrademarks",  ""
 
 
diff --git a/SelfServiceCommon/Massai/cpp/BillingService/inc/BillingS_def.h b/SelfServiceCommon/Massai/cpp/BillingService/inc/BillingS_def.h
index c81e8a69..d4dadb52 100644
--- a/SelfServiceCommon/Massai/cpp/BillingService/inc/BillingS_def.h
+++ b/SelfServiceCommon/Massai/cpp/BillingService/inc/BillingS_def.h
@@ -1,39 +1,48 @@
 #define FILE_VER_BIN     1,0,0,8
-#define FILE_VER_STR     "FileVersion", "1.0.0.8\0"
+#define FILE_VER_STR     "FileVersion", "1.0.0.8"
 
 /*!
-\page Billing.exe Billing Service - Release Notes
+\page Billing.exe Release Notes - Billing.exe - Billing Service
 
-\date 16.07.2013
-\li V1.0.0.8
-\li Mantis 0026690: Billing: add attribute Brand to Billing-Data
+@version 1.0.0.8
 
+\section changehistory Change history
 
-\date 25.04.2013 \li V1.0.0.7
-                 \li Mantis 0026967: BillingService: recognize non print AEA messages
+Billing.exe
 
-\date 07.11.2012 \li V1.0.0.6
-                 \li SteT Mantis 0024999: service creates very large files (VC10)
+\version 1.0.0.8
+\li 16.07.2013
+\li SteT Mantis 0026690: Billing: add attribute Brand to Billing-Data
 
-\date 16.12.2011 \li V1.0.0.5
-                 \li SteT Mantis 0021947: mstarter: problems at shutdown
-                     shutdown on SIGBREAK, ignore CTRL_LOGOFF_EVENT
+\version 1.0.0.7
+\li 25.04.2013
+\li SteT Mantis 0026967: BillingService: recognize non print AEA messages
 
-\date 27.04.2006 \li V 1.0.0.4
-                 \li SteT EDB 26755: BillingService parsed die Antwort vom GPP nicht
+\version 1.0.0.6
+\li 07.11.2012
+\li SteT Mantis 0024999: service creates very large files (VC10)
 
-\date 27.04.2006 \li V 1.0.0.3
-                 \li SteT EDB 26729: Remove CTRL_BREAK
+\version 1.0.0.5
+\li 16.12.2011
+\li SteT Mantis 0021947: mstarter: problems at shutdown\n
+  shutdown on SIGBREAK, ignore CTRL_LOGOFF_EVENT
 
-\date 25.10.2006 \li V1.0.0.2
-                 \li SteT EDB 26693: Abgleich des neuen CC Streams SelfServicePlatform mit den alten Streams
-                 \li limit trace output
+\version 1.0.0.4
+\li 27.04.2006
+\li SteT EDB 26755: BillingService parsed die Antwort vom GPP nicht
 
- */
+\version 1.0.0.3
+\li 27.04.2006
+\li SteT EDB 26729: Remove CTRL_BREAK
 
-#define FILEDESCRIPTION  "FileDescription"  ,"Billing Service\0"
-#define INTERNALNAME     "InternalName"     ,"Billing Service\0"
-#define LEGALTRADEMARKS  "LegalTrademarks"  ,"\0"
-#define ORIGINALFILENAME "OriginalFilename" ,"Billing.exe\0"
+\version 1.0.0.2
+\li 25.10.2006
+\li SteT EDB 26693: Abgleich des neuen CC Streams SelfServicePlatform mit den alten Streams
+\li limit trace output
 
+ */
 
+#define FILEDESCRIPTION  "FileDescription",  "Billing Service"
+#define INTERNALNAME     "InternalName",     "Billing.exe"
+#define ORIGINALFILENAME "OriginalFilename", "Billing.exe"
+#define LEGALTRADEMARKS  "LegalTrademarks",  ""
diff --git a/SelfServiceCommon/Massai/cpp/Controller/inc/mstarter_def.h b/SelfServiceCommon/Massai/cpp/Controller/inc/mstarter_def.h
index fa420ab7..d7e5512e 100644
--- a/SelfServiceCommon/Massai/cpp/Controller/inc/mstarter_def.h
+++ b/SelfServiceCommon/Massai/cpp/Controller/inc/mstarter_def.h
@@ -1,176 +1,181 @@
-/**
- *  @file      mstarter_def.h
- *
- *  @brief     Version constants
- *
- *  @date      21.03.2006
- *
- *  @author    Materna Information & Communications (MaM)
- *
- ************************************************************************/
-
 #define FILE_VER_BIN     1,1,1,8
-#define FILE_VER_STR     "FileVersion", "1.1.1.8\0"
+#define FILE_VER_STR     "FileVersion", "1.1.1.8"
 
 /*!
-\page massaistarter mStarter - Release Notes
+\page mstarter.exe Release Notes - mstarter.exe - process manager
 
 @version 1.1.1.8
 
 \section change1 Change history
 
-\date 24.07.2013 \li V1.1.1.8
-                 \li extension - JK Mantis 24145
-                 \li Port range is now hardcoded to 20050 - 20199
-                 \li (Thomas G wanted some ports for himself)
+mstarter.exe
+
+\version 1.1.1.8
+\li 24.07.2013
+\li extension - JK Mantis 24145
+\li Port range is now hardcoded to 20050 - 20199
+\li (Thomas G wanted some ports for himself)
 
-\date 24.07.2013 \li V1.1.1.7
-                 \li extension - JK Mantis 24145 Remove CORBA.SERVANTS configuration
-                 \li - activate SUPPORT_SERVANTPORT_LIST
-                 \li - scan not only the items configured foor execution in
-                 \li   PLATFORM.EXEC, but all entries there. The $SERVANTPORT_LIST will
-                 \li   contain $SERVANTPORT mentioned there too.
+\version 1.1.1.7
+\li 24.07.2013
+\li extension - JK Mantis 24145 Remove CORBA.SERVANTS configuration
+\li - activate SUPPORT_SERVANTPORT_LIST
+\li - scan not only the items configured foor execution in
+\li   PLATFORM.EXEC, but all entries there. The $SERVANTPORT_LIST will
+\li   contain $SERVANTPORT mentioned there too.
 
-\date 04.07.2013
-\li V1.1.1.6
+\version 1.1.1.6
+\li 04.07.2013
 \li Mantis 0027896: mstarter: optimize start order
 
-\date 27.06.2013
-\li V1.1.1.5
+\version 1.1.1.5
+\li 27.06.2013
 \li Mantis 0026902: Internet Explorer Process doesn't close
-closing child processes, except process of exlusionList
-read environment variables and configuration entry PLATFORM.ENVIRONMENT.Variables for substitution
-new configuration item:
-  PLATFORM.ENVIRONMENT.Variables
-  type: text list
+\li closing child processes, except process of exlusionList
+\li read environment variables and configuration entry PLATFORM.ENVIRONMENT.Variables for substitution
+\li new configuration item:\n
+  PLATFORM.ENVIRONMENT.Variables\n
+  type: text list\n
   description: substitution variables; format variablename=value
-
-TerminationExlusion is used for exlusionList in process termination if nstarter
-
-
-\date 14.08.2012 \li V1.1.1.3
-                 \li extension - JK Mantis 24145
-                      - Port range is now hardcoded to 20002 - 20199
-                        ( Configuration can be enabled
-                          with ALLOW_TO_CONFIGURE_PORT_RANGE ).
-                      - CORBA.SERVANTS.<servant> is no longer set
-                        ( Option can be enabled with SETUP_CORBA_SERVANTS ).
-                        A separate tool setCORBASERVANTS can be
-                        configured in PLATFORM.EXEC for legacy apps.
-
-\date 10.08.2012 \li V1.1.1.2
-                 \li bugfix - SteT Mantis 0024162: Suspicious assignment
-
-\date 10.08.2012 \li V1.1.1.1
-                 \li extension - JK Mantis 24145 Remove CORBA.SERVANTS configuration
-
-\date 16.12.2011 \li V1.1.1.0
-                 \li SteT Mantis 0021947: mstarter: problems at shutdown
-
-   fixed problems in mstarter:
-         - listenOnChange can be activated without restarting mstarter:
-            listen on items:
-              SERVICES.STARTER.listenOnConfigChange
-              PLATFORM.HARDWARE.Devices
-              PLATFORM.SOFTWARE.Services
-              PLATFORM.EXEC.*
-         - listenOnChange works for processes without configured WND
-             send ctrl-break to console applications (console ist started as new process group)
-             first close visible windows and then all windows for process without option WND
-         - option WNDCLASS added for PLATFORM.EXEC.* items to find process main window by class
-         - mstarter arguments changed:
-             WatchDog-Timeout must be set with ":"
-             semaphore name is optional, if not set mstarter waits for ctrl-c, ctrl-break
-             mstarter.exe [<StopSemaphoreName>][-server][-cfg][-watchDog:<ms>]
-               StopSemaphoreName: name of event semaphore, posted to stop
-               server: start in server mode
-               cfg: create configuraton items
-               watchdog: timeout for restart of processes; -1 deactivates restart
-         - watchdog:
-              process are watched all the time
-              more information of a process are printed at process terminated
-         - shutdown:
-              when all processes are terminated, mstarter terminated
-              mstarter can be stopped while starting processes
-         - progress bar:
-              progress is calculated by mstarter; depends on mcontrol V1.0.2.0
-
-
-\date 17.11.2011 \li V1.1.0.8
-                 \li GiT: Mantis0021563: SelfServiceCommon Controller changes for VC10 compatibilty
-
-\date 12.09.2011 \li V 1.1.0.7
-                 \li GiT: Mantis0014919: placeholder for system paths
-                 \li      The following substitutions are now supported:
-                 \li      - $PROGRAMFILES will be substituted by the environment var ProgramFiles
-                 \li      - $SYSTEMROOT will be substituted by the environment var SystemRoot
-                 \li      - $SYSTEMDRIVE will be substituted by the environment var SystemDrive
-
-\date 25.01.2010 \li V 1.1.0.6
-                 \li bufix - Wrong illegal pointer dereference - crash (JK)
-
-\date 08.12.2009 \li V 1.1.0.5
-                 \li feature - JK MANTIS0011257: mcontrol reboot wird nicht ausgeführt, plattform wird nicht gestoppt
-                 \li           - More traces and try-catch blocks to handle unexpected situations.
-                 \li           - Increase use of systools functions.
-
-\date 27.10.2009 \li V 1.1.0.4
-                 \li enhancement - SteT MANTIS 0010640: Plattform-Konfiguration PLATFORM.EXEC set length to 9000 (some Execs are longer than 255)
-
-\date 26.05.2009 \li V 1.1.0.3
-                 \li enhancement - SteT MANTIS 0006873: description for configuration items under PLATFORM.EXEC improved
-                 \li                       allow " in parameter
-
-\date 26.11.2008 \li V 1.1.0.2
-                 \li bufix - creating correct default entries for server mode (PtM)
-
-\date 21.11.2008 \li V 1.1.0.1
-                 \li bufix - catch exceptions during config access (PtM)
-
-\date 20.11.2008 \li V 1.1.0.0 (PtM)
-                 \li bugfix - process count bar display now the correct number of running applications
-                 \li bugfix - removed memory leaks (configuration)
-                 \li feature - with the configuration item "SERVICES.STARTER.listenOnConfigChange" the feature
-                               "listen on configuration changes" can be activated.
-                               This feature starts and stops processes if they have been added or removed under:
-                               PLATFORM.SOFTWARE.Services or PLATFORM.HARDWARE.Devices
-
-\date 02.06.2008 \li V 1.0.5.7
-                 \li SteT EDB 28044: wrong working directory used when the program parameters contain \
-                 \li     now the parameters are ignored when the working directory is extracted from command line if the parameter are enclosed in double quotes
-
-\date 07.03.2008 \li V 1.0.5.6
-                 \li SteT closed unused handles, read configuration once at startup (memory leak)
-
-\date 07.08.2007 \li V 1.0.5.5
-                 \li SteT EDB 27719: evaluate all parameter values from EXEC configuration
-
-\date 29.03.2007 \li V 1.0.5.4
-                 \li bugfix display correct process bar after watchdog restarted a process(PtM)
-
-\date 24.01.2007 \li V 1.0.5.3
-                 \li SteT Send CTRL_C and CTRL_BREAK
-
-\date 07.11.2006 \li V 1.0.5.2
-                 \li SteT EDB 26729: Remove CTRL_BREAK
-
-
-\date 27.04.2006 \li V 1.0.5.1  BschR
-                 \li change startup order from: devices -> services -> platform to: services -> platform -> devices
-
-\date 21.03.2006 \li V 1.0.5.0  MaM
-                 \li new feature: watchDog (default: ON, PollTime 1000 ms, no selective Processlist)
-                 \li to switch off watchDog call: mstarter.exe <sema> [-server] -watchDog -1
-                 \li to to set watchDog poll intervall call: mstarter.exe <sema> [-server] -watchDog <ms>
-
+\li TerminationExlusion is used for exlusionList in process termination if nstarter
+
+\version 1.1.1.3
+\li 14.08.2012
+\li extension - JK Mantis 24145
+\li Port range is now hardcoded to 20002 - 20199\n
+  ( Configuration can be enabled\n
+    with ALLOW_TO_CONFIGURE_PORT_RANGE ).
+\li CORBA.SERVANTS.<servant> is no longer set\n
+  ( Option can be enabled with SETUP_CORBA_SERVANTS ).\n
+  A separate tool setCORBASERVANTS can be\n
+  configured in PLATFORM.EXEC for legacy apps.
+
+\version 1.1.1.2
+\li 10.08.2012
+\li bugfix - SteT Mantis 0024162: Suspicious assignment
+
+\version 1.1.1.1
+\li 10.08.2012
+\li extension - JK Mantis 24145 Remove CORBA.SERVANTS configuration
+
+\version 1.1.1.0
+\li 16.12.2011
+\li SteT Mantis 0021947: mstarter: problems at shutdown
+\li   fixed problems in mstarter:
+\li listenOnChange can be activated without restarting mstarter:\n
+   listen on items:\n
+     SERVICES.STARTER.listenOnConfigChange\n
+     PLATFORM.HARDWARE.Devices\n
+     PLATFORM.SOFTWARE.Services\n
+     PLATFORM.EXEC.*
+\li listenOnChange works for processes without configured WND\n
+    send ctrl-break to console applications (console ist started as new process group)\n
+    first close visible windows and then all windows for process without option WND
+\li option WNDCLASS added for PLATFORM.EXEC.* items to find process main window by class
+\li mstarter arguments changed:\n
+    WatchDog-Timeout must be set with ":"\n
+    semaphore name is optional, if not set mstarter waits for ctrl-c, ctrl-break\n
+    mstarter.exe [<StopSemaphoreName>][-server][-cfg][-watchDog:<ms>]\n
+      StopSemaphoreName: name of event semaphore, posted to stop\n
+      server: start in server mode\n
+      cfg: create configuraton items\n
+      watchdog: timeout for restart of processes; -1 deactivates restart
+\li watchdog:\n
+     process are watched all the time\n
+     more information of a process are printed at process terminated
+\li shutdown:\n
+     when all processes are terminated, mstarter terminated\n
+     mstarter can be stopped while starting processes
+\li progress bar:\n
+     progress is calculated by mstarter; depends on mcontrol V1.0.2.0
+
+\version 1.1.0.8
+\li 17.11.2011
+\li GiT: Mantis0021563: SelfServiceCommon Controller changes for VC10 compatibilty
+
+\version  1.1.0.7
+\li 12.09.2011
+\li GiT: Mantis0014919: placeholder for system paths
+\li      The following substitutions are now supported:
+\li      - $PROGRAMFILES will be substituted by the environment var ProgramFiles
+\li      - $SYSTEMROOT will be substituted by the environment var SystemRoot
+\li      - $SYSTEMDRIVE will be substituted by the environment var SystemDrive
+
+\version  1.1.0.6
+\li 25.01.2010
+\li bufix - Wrong illegal pointer dereference - crash (JK)
+
+\version  1.1.0.5
+\li 08.12.2009
+\li feature - JK MANTIS0011257: mcontrol reboot wird nicht ausgeführt, plattform wird nicht gestoppt
+\li           - More traces and try-catch blocks to handle unexpected situations.
+\li           - Increase use of systools functions.
+
+\version  1.1.0.4
+\li 27.10.2009
+\li enhancement - SteT MANTIS 0010640: Plattform-Konfiguration PLATFORM.EXEC set length to 9000 (some Execs are longer than 255)
+
+\version  1.1.0.3
+\li 26.05.2009
+\li enhancement - SteT MANTIS 0006873: description for configuration items under PLATFORM.EXEC improved
+\li                       allow " in parameter
+
+\version  1.1.0.2
+\li 26.11.2008
+\li bufix - creating correct default entries for server mode (PtM)
+
+\version  1.1.0.1
+\li 21.11.2008
+\li bufix - catch exceptions during config access (PtM)
+
+\version  1.1.0.0
+\li 20.11.2008
+\li (PtM) bugfix - process count bar display now the correct number of running applications
+\li bugfix - removed memory leaks (configuration)
+\li feature - with the configuration item "SERVICES.STARTER.listenOnConfigChange" the feature\n
+    "listen on configuration changes" can be activated.\n
+    This feature starts and stops processes if they have been added or removed under:\n
+    PLATFORM.SOFTWARE.Services or PLATFORM.HARDWARE.Devices
+
+\version  1.0.5.7
+\li 02.06.2008
+\li SteT EDB 28044: wrong working directory used when the program parameters contain \
+\li     now the parameters are ignored when the working directory is extracted from command line if the parameter are enclosed in double quotes
+
+\version  1.0.5.6
+\li 07.03.2008
+\li SteT closed unused handles, read configuration once at startup (memory leak)
+
+\version  1.0.5.5
+\li 07.08.2007
+\li SteT EDB 27719: evaluate all parameter values from EXEC configuration
+
+\version  1.0.5.4
+\li 29.03.2007
+\li bugfix display correct process bar after watchdog restarted a process(PtM)
+
+\version  1.0.5.3
+\li 24.01.2007
+\li SteT Send CTRL_C and CTRL_BREAK
+
+\version  1.0.5.2
+\li 07.11.2006
+\li SteT EDB 26729: Remove CTRL_BREAK
+
+\version  1.0.5.1
+\li 27.04.2006
+\li BschR change startup order from: devices -> services -> platform to: services -> platform -> devices
+
+\version  1.0.5.0
+\li 21.03.2006
+\li MaM new feature: watchDog (default: ON, PollTime 1000 ms, no selective Processlist)
+\li to switch off watchDog call: mstarter.exe <sema> [-server] -watchDog -1
+\li to to set watchDog poll intervall call: mstarter.exe <sema> [-server] -watchDog <ms>
 */
 
-#define IDI_ICON1 101
-#define IDI_ICON2 102
-
-#define FILEDESCRIPTION  "FileDescription"  ,"mstarter\0"
-#define INTERNALNAME     "InternalName"     ,"mstarter\0"
-#define LEGALTRADEMARKS  "LegalTrademarks"  ,"\0"
-#define ORIGINALFILENAME "OriginalFilename" ,"mstarter.exe\0"
+#define FILEDESCRIPTION  "FileDescription",  "mstarter"
+#define INTERNALNAME     "InternalName",     "mstarter.exe"
+#define LEGALTRADEMARKS  "LegalTrademarks",  ""
+#define ORIGINALFILENAME "OriginalFilename", "mstarter.exe"
 
 
diff --git a/SelfServiceCommon/Massai/cpp/GracefullyCrash/inc/local_def_gracefullycrash.h b/SelfServiceCommon/Massai/cpp/GracefullyCrash/inc/local_def_gracefullycrash.h
index 95e64ce4..e7ebed60 100644
--- a/SelfServiceCommon/Massai/cpp/GracefullyCrash/inc/local_def_gracefullycrash.h
+++ b/SelfServiceCommon/Massai/cpp/GracefullyCrash/inc/local_def_gracefullycrash.h
@@ -1,17 +1,21 @@
-/*! \page gracefullycrash gracefullycrash library - Release Notes
- *
- *
- *  \date 28.06.2007 \li V1.0.0.0
- *                   \li SteT EDB 27474: System MsgBox
- *
- */
-
 #define FILE_VER_BIN     1,0,0,0
-#define FILE_VER_STR     "FileVersion", "1.0.0.0\0"
+#define FILE_VER_STR     "FileVersion", "1.0.0.0"
+
+/*!
+\page gracefullycrash.lib ReleaseNotes - gracefullycrash.lib
+
+\section changehistory Change history
+
+@version 1.0.0.0
 
-#define FILEDESCRIPTION  "FileDescription"  ,"GracefullyCrash\0"
-#define INTERNALNAME     "InternalName"     ,"GracefullyCrash\0"
-#define LEGALTRADEMARKS  "LegalTrademarks"  ,"\0"
-#define ORIGINALFILENAME "OriginalFilename" ,"GracefullyCrash.lib\0"
+gracefullycrash.lib
 
+\version 1.0.0.0
+\li 28.06.2007
+\li SteT EDB 27474: System MsgBox
+*/
 
+#define FILEDESCRIPTION  "FileDescription",  "gracefullycrash"
+#define INTERNALNAME     "InternalName",     "gracefullycrash.lib"
+#define ORIGINALFILENAME "OriginalFilename", "gracefullycrash.lib"
+#define LEGALTRADEMARKS  "LegalTrademarks",  ""
diff --git a/SelfServiceCommon/Massai/cpp/Graphic/src/local_def.cpp b/SelfServiceCommon/Massai/cpp/Graphic/src/local_def.cpp
index 8c4ac14d..1bb14e9d 100644
--- a/SelfServiceCommon/Massai/cpp/Graphic/src/local_def.cpp
+++ b/SelfServiceCommon/Massai/cpp/Graphic/src/local_def.cpp
@@ -1,33 +1,37 @@
-#include "Tools/VERSIONLIB.h"
+#include "tools/versionlib.h"
 
 VERSION_ID_DT(graphic, 1, 0, 0, 4);
 
 /*!
-\page graphic graphic library - Release Notes
+\page massai_graphic.lib Release Notes massai_graphic.lib
 
-\date 13.08.2013
-\li V1.0.0.4
+@version 1.0.0.4
+
+\section changehistory Change history
+
+massai_graphic.lib
+
+\version 1.0.0.4
+\li 13.08.2013
 \li Mantis 0028572: GPPDev: Receipt support Barcode according to AEA2011
-added field size_t columns to BarcodePDF417Settings
-if rows != 0 use rows
-if columns != 0 use columns
-else use ratio of height/width = 3/4
+\li added field size_t columns to BarcodePDF417Settings\n
+if rows != 0 use rows\n
+if columns != 0 use columns\n
+else use ratio of height/width = 3/4\n
 
-\date 09.07.2013
-\li V1.0.0.3
+\version 1.0.0.3
+\li 09.07.2013
 \li Mantis 0027971: GPPDEV: barcode 128 wrong checkdigit for Type B and Type C
 
-\date 27.11.2012
-\li V1.0.0.2
+\version 1.0.0.2
+\li 27.11.2012
 \li SteT Mantis 0024715: Build: VC10; fix compiler warnings
 
-\date 25.02.2011
-\li V1.0.0.1
+\version 1.0.0.1
+\li 25.02.2011
 \li SteT Bugfix Interleaved2of5 ignore invalid character
 
-\date 19.10.2010
-\li V1.0.0.0
+\version 1.0.0.0
+\li 19.10.2010
 \li SteT Initial Version
-
 */
-
diff --git a/SelfServiceCommon/Massai/cpp/MassaiCorba/inc/localclnt_def.h b/SelfServiceCommon/Massai/cpp/MassaiCorba/inc/localclnt_def.h
index aace85f6..2fcdb867 100644
--- a/SelfServiceCommon/Massai/cpp/MassaiCorba/inc/localclnt_def.h
+++ b/SelfServiceCommon/Massai/cpp/MassaiCorba/inc/localclnt_def.h
@@ -1,9 +1,22 @@
 #define FILE_VER_BIN     1,0,0,0
-#define FILE_VER_STR     "FileVersion", "1.0.0.0\0"
+#define FILE_VER_STR     "FileVersion", "1.0.0.0"
 
-#define FILEDESCRIPTION  "FileDescription"  ,"Corba Testclient\0"
-#define INTERNALNAME     "InternalName"     ,"Testclient\0"
-#define LEGALTRADEMARKS  "LegalTrademarks"  ,"\0"
-#define ORIGINALFILENAME "OriginalFilename" ,"testclnt.exe\0"
+/*!
+\page testclnt.exe Release Notes - testclnt.exe - test client for mcorba.lib
+
+@version 1.0.0.0
+
+\section changehistory Change history
+
+testclnt.exe
+
+\version 1.0.0.0
+\li initial version
+*/
+
+#define FILEDESCRIPTION  "FileDescription",  "Corba Testclient"
+#define INTERNALNAME     "InternalName",     "testclnt.exe"
+#define LEGALTRADEMARKS  "LegalTrademarks",  ""
+#define ORIGINALFILENAME "OriginalFilename", "testclnt.exe"
 
 
diff --git a/SelfServiceCommon/Massai/cpp/MassaiCorba/inc/localsrv_def.h b/SelfServiceCommon/Massai/cpp/MassaiCorba/inc/localsrv_def.h
index 8b75f4c0..c5f81ef6 100644
--- a/SelfServiceCommon/Massai/cpp/MassaiCorba/inc/localsrv_def.h
+++ b/SelfServiceCommon/Massai/cpp/MassaiCorba/inc/localsrv_def.h
@@ -1,9 +1,22 @@
 #define FILE_VER_BIN     1,0,0,0
-#define FILE_VER_STR     "FileVersion", "1.0.0.0\0"
+#define FILE_VER_STR     "FileVersion", "1.0.0.0"
 
-#define FILEDESCRIPTION  "FileDescription"  ,"Corba Test Servant\0"
-#define INTERNALNAME     "InternalName"     ,"Testservant\0"
-#define LEGALTRADEMARKS  "LegalTrademarks"  ,"\0"
-#define ORIGINALFILENAME "OriginalFilename" ,"testsrvr.exe\0"
+/*!
+\page testsrvr.exe Release Notes - testsrvr.exe - test servant for mcorba.lib
+
+@version 1.0.0.0
+
+\section changehistory Change history
+
+testsrvr.exe
+
+\version 1.0.0.0
+\li initial version
+*/
+
+#define FILEDESCRIPTION  "FileDescription",  "Corba Test Servant"
+#define INTERNALNAME     "InternalName",     "testsrvr.exe"
+#define LEGALTRADEMARKS  "LegalTrademarks",  ""
+#define ORIGINALFILENAME "OriginalFilename", "testsrvr.exe"
 
 
diff --git a/SelfServiceCommon/Massai/cpp/MassaiCorba/inc/massaiCORBA_def.h b/SelfServiceCommon/Massai/cpp/MassaiCorba/inc/massaiCORBA_def.h
index 75ee719a..bc7c6758 100644
--- a/SelfServiceCommon/Massai/cpp/MassaiCorba/inc/massaiCORBA_def.h
+++ b/SelfServiceCommon/Massai/cpp/MassaiCorba/inc/massaiCORBA_def.h
@@ -1,155 +1,146 @@
-/**
- *  @file      massaiCORBA_def.h
- *
- *  @brief     Version file
- *
- *  @date      09.05.2006
- *
- *  @author    Materna Information & Communications (MaM)
- *
- ************************************************************************/
-
-// #define FILE_VER_BIN     1,0,6,20
-// #define FILE_VER_STR     "FileVersion", "1.0.6.20\0"
-
 /*!
-\page mcorba.lib, mcorbas.lib Massai CORBA library - Release Notes
+\page mcorba.lib Release Notes - mcorba.lib - Massai CORBA library
 
 @version 1.0.6.20
 
-\section change1 Change history
-
-\date 27.11.2012 \li V1.0.6.20
-                 \li SteT - enhancement, Mantis 0024715: Build: VC10; fix compiler warnings
-
-\date 23.08.2012 \li V1.0.6.19
-                 \li SteT - bugfix, Mantis 24158 call ACE::init, ACE::fini around use of ORB_init, ORB::destroy
-                            bugfix, use correct POA for servant deactivation
-                            bugfix, fixed reading port after hostname in configuration from CORBA.SERVANTS.*
-                            bugfix, with ORBClientConnectionHandler RW, ACE TAO ignores CloseConnection-Messages
-                                    use MT_NOUPCALL instead
-
-\date 13.03.2012 \li V1.0.6.18
-                 \li SteT - enhancement, Mantis 0018942: mCorba: creation order of servant orb and client orb and synchronisation problems
-
-                 \li environment settings:
-                    CORBA_AUTOPOLL_ENABLE     enable auto polling, client only
-                    CORBA_AUTOPOLL_DISABLE    disable auto polling, client only
-                    CORBA_AUTOPOLL_TIMEOUT    poll timeout in ms, default 60000 ms, client only
-                    CORBA_INVOCATION_TIMEOUT  default invocation timeout in ms, default 60000ms
-                    ORBENDPOINT_PORT          default orb endpoint port
-                    CORBA_NUMERIC_ORB         use ip instead of hostname in ior
-                    CORBA_ENABLE_LOGGING      enable ORB logging
-                    CORBA_USE_NESTED_CALLS    allow nested calls: use the threads in client calls for processing servant calls
-
-                 \li logging
-                    TAO ORB logging is written to stderr
-                    massaiCORBA logging is written with OutputDebugString
-
-                 \li localhost
-                    cfg: empty host in configuration (CORBA.SERVANTS.*) is mapped to localhost
-                    servant creation: localhost is mapped to 0.0.0.0
-                    client connecting servant: localhost is mapped to local hostname
-                    to bind and connect to 127.0.0.1 use 127.0.0.1 for hostname
-
-                 \li multiple network interfaces
-                    the hostname in the ior is set to the local hostname, if the servant is listening on all interfaces (interface = localhost or no interface)
-                    to use multiple hostnames in the ior use the interface = 0.0.0.0
-
-                 \li multiple orbs
-                    every module creates his own orbs (avoid interaction between orb creation in dlls and executable)
-                    for every ORBEndpoint a new orb is created
-                    clients use the first servant orb or an own client orb, if no servant orb is created
-                    for implicit activation the first servant orb is used
-                       warning: this is not module independent, all modules use the same orb for implicit activation
-                                the first servant orb of the module that creates its first servant orb at last is used
-                                => avoid creating servants in dlls
-
-\date 09.06.2011 \li V1.0.6.17
-                 \li GiT, rollback of previous change
-
-\date 09.06.2011 \li V1.0.6.16
-                 \li GiT, modification: changed orb connection handler behaviour to RW
-                          TAO description : RW selects a strategy that simply blocks in recv() when waiting for a response from the server
-                                            instead of waiting in the Reactor using the Leader/Followers pattern.
-                          This change should prevent nested upcalls.
-                     To change the behaviour back to the old strategy set the environment variable
-                     STANDARD_ORB_WAIT_STRATEGY
-                 \li QA-Tests: All platform modules including the connection to applications are affected, a thorough integration test is needed
-
-\date 21.04.2011 \li V1.0.6.15
-                 \li SteT, enhancement, added trace to corbaorb
-
-\date 28.10.2009 \li V1.0.6.14
-                 \li SteT Mantis 11665: deactivate CorbaPoll (per default)
-
-   per default the corbaClient is created with the parameter AUTO_POLL_OFF
-   this behaviour can be changed with environment setting:
+\section changehistory Change history
+
+mcorba.lib
+
+\version 1.0.6.20
+\li 27.11.2012
+\li SteT - enhancement, Mantis 0024715: Build: VC10; fix compiler warnings
+
+\version 1.0.6.19
+\li 23.08.2012
+\li SteT - bugfix, Mantis 24158 call ACE::init, ACE::fini around use of ORB_init, ORB::destroy
+\li bugfix, use correct POA for servant deactivation
+\li bugfix, fixed reading port after hostname in configuration from CORBA.SERVANTS.*
+\li bugfix, with ORBClientConnectionHandler RW, ACE TAO ignores CloseConnection-Messages\n
+            use MT_NOUPCALL instead
+
+\version 1.0.6.18
+\li 13.03.2012
+\li SteT - enhancement, Mantis 0018942: mCorba: creation order of servant orb and client orb and synchronisation problems
+\li environment settings:\n
+    CORBA_AUTOPOLL_ENABLE     enable auto polling, client only\n
+    CORBA_AUTOPOLL_DISABLE    disable auto polling, client only\n
+    CORBA_AUTOPOLL_TIMEOUT    poll timeout in ms, default 60000 ms, client only\n
+    CORBA_INVOCATION_TIMEOUT  default invocation timeout in ms, default 60000ms\n
+    ORBENDPOINT_PORT          default orb endpoint port\n
+    CORBA_NUMERIC_ORB         use ip instead of hostname in ior\n
+    CORBA_ENABLE_LOGGING      enable ORB logging\n
+    CORBA_USE_NESTED_CALLS    allow nested calls: use the threads in client calls for processing servant calls
+\li logging\n
+    TAO ORB logging is written to stderr\n
+    massaiCORBA logging is written with OutputDebugString
+\li localhost\n
+    cfg: empty host in configuration (CORBA.SERVANTS.*) is mapped to localhost\n
+    servant creation: localhost is mapped to 0.0.0.0\n
+    client connecting servant: localhost is mapped to local hostname\n
+    to bind and connect to 127.0.0.1 use 127.0.0.1 for hostname
+\li multiple network interfaces\n
+    the hostname in the ior is set to the local hostname, if the servant is listening on all interfaces (interface = localhost or no interface)\n
+    to use multiple hostnames in the ior use the interface = 0.0.0.0
+\li multiple orbs\n
+    every module creates his own orbs (avoid interaction between orb creation in dlls and executable)\n
+    for every ORBEndpoint a new orb is created\n
+    clients use the first servant orb or an own client orb, if no servant orb is created\n
+    for implicit activation the first servant orb is used\n
+       warning: this is not module independent, all modules use the same orb for implicit activation\n
+                the first servant orb of the module that creates its first servant orb at last is used\n
+                => avoid creating servants in dlls
+
+\version 1.0.6.17
+\li 09.06.2011
+\li GiT, rollback of previous change
+
+\version 1.0.6.16
+\li 09.06.2011
+\li GiT, modification: changed orb connection handler behaviour to RW\n
+    TAO description : RW selects a strategy that simply blocks in recv() when waiting for a response from the server\n
+                      instead of waiting in the Reactor using the Leader/Followers pattern.\n
+    This change should prevent nested upcalls.\n
+    To change the behaviour back to the old strategy set the environment variable\n
+    STANDARD_ORB_WAIT_STRATEGY
+\li QA-Tests: All platform modules including the connection to applications are affected, a thorough integration test is needed
+
+\version 1.0.6.15
+\li 21.04.2011
+\li SteT, enhancement, added trace to corbaorb
+
+\version 1.0.6.14
+\li 28.10.2009
+\li SteT Mantis 11665: deactivate CorbaPoll (per default)\n
+   per default the corbaClient is created with the parameter AUTO_POLL_OFF\n
+   this behaviour can be changed with environment setting:\n
       use CORBA_AUTOPOLL_ENABLE to enabling the polling for non local connections
 
-\date 28.04.2009 \li V1.0.6.13
-                 \li SteT bugfix: use killClientOrb when starting new Servant to avoid LOCATION_FORWARD from servant port to client port
-
-\date 04.12.2008 \li V1.0.6.12
-                 \li SteT Mantis 6871: correct CorbaPoll Tracelevel
-
-\date 04.09.2008 \li V1.0.6.11
-                 \li SteT EDB 28111: Invalid Pointer Read / Free Memory Read in CorbaPoll
-
-
-\date 17.07.2008 \li V1.0.6.10
-                 \li SteT Change: corbaClient sends KeepAlive requests
-
- corbaClient is derived from corbaPoll and the default behaviour of corbaClient ist changed:
-
-   per default the corbaClient is created with the parameter AUTO_POLL_NON_LOCAL
-   if the servant is not obvious on the same machine, the client send every minute a LocateRequest message to the servant
-
-   the behaviour can be changed with the autopoll parameter in the constructor
-   and with environment setting:
-      use CORBA_AUTOPOLL_DISABLE to disable the polling
+\version 1.0.6.13
+\li 28.04.2009
+\li SteT bugfix: use killClientOrb when starting new Servant to avoid LOCATION_FORWARD from servant port to client port
+
+\version 1.0.6.12
+\li 04.12.2008
+\li SteT Mantis 6871: correct CorbaPoll Tracelevel
+
+\version 1.0.6.11
+\li 04.09.2008
+\li SteT EDB 28111: Invalid Pointer Read / Free Memory Read in CorbaPoll
+
+\version 1.0.6.10
+\li 17.07.2008
+\li SteT Change: corbaClient sends KeepAlive requests
+\li corbaClient is derived from corbaPoll and the default behaviour of corbaClient ist changed:\n
+   per default the corbaClient is created with the parameter AUTO_POLL_NON_LOCAL\n
+   if the servant is not obvious on the same machine, the client send every minute a LocateRequest message to the servant\n
+   the behaviour can be changed with the autopoll parameter in the constructor\n
+   and with environment setting:\n
+      use CORBA_AUTOPOLL_DISABLE to disable the polling\n
       use CORBA_AUTOPOLL_TIMEOUT to set the polling timeout (in ms)
+\li a new corbaClient constructor is added for callback-listener set by remote (the KeepAlive mechanism is not used if the listener is used without the corbaClient)
 
- a new corbaClient constructor is added for callback-listener set by remote (the KeepAlive mechanism is not used if the listener is used without the corbaClient)
-
+\version 1.0.6.9
+\li 12.11.2007
+\li SteT Bugfix unresolved external of CfgXMgrB (ACE_NO_INLINE removed)
 
-\date 12.11.2007 \li V1.0.6.9
-                 \li SteT Bugfix unresolved external of CfgXMgrB (ACE_NO_INLINE removed)
+\version 1.0.6.8
+\li 25.07.2007
+\li SteT EDB 26866: the classes corbaClient, corbaServantEx stores the ior and releases the memory used
+\li     the method ior() returns const char* (was char*)
+\li     the caller is not responsible for releasing the returned buffer any more
 
-\date 25.07.2007 \li V1.0.6.8
-                 \li SteT EDB 26866: the classes corbaClient, corbaServantEx stores the ior and releases the memory used
-                 \li     the method ior() returns const char* (was char*)
-                 \li     the caller is not responsible for releasing the returned buffer any more
+\version 1.0.6.7
+\li 20.06.2007
+\li SteT EDB 27607: wrong constructor results in access violation
 
-\date 20.06.2007 \li V1.0.6.7
-                 \li SteT EDB 27607: wrong constructor results in access violation
+\version 1.0.6.6
+\li 20.06.2007
+\li SteT EDB 27562: Pure virtual function call, after removeServant
 
-\date 20.06.2007 \li V1.0.6.6
-                 \li SteT EDB 27562: Pure virtual function call, after removeServant
+\version 1.0.6.5
+\li 30.05.2007
+\li feature - hostname_in_ior can be set from external. PtM
 
-\date 30.05.2007 \li V1.0.6.5
-                 \li feature - hostname_in_ior can be set from external. PtM
+\version 1.0.6.4 MaM
+\li 17.01.2007
+\li MaM EDB 27073: mCorba / mDisplay: kein CORBA Roundtrip TO bei Fremdplattform
 
-\date 17.01.2007 \li V1.0.6.4 MaM
-                 \li MaM EDB 27073: mCorba / mDisplay: kein CORBA Roundtrip TO bei Fremdplattform
+\version 1.0.6.3 SteT
+\li 09.08.2006
+\li SteT EDB 26406: Erweiterung der massaiCORBA Funktionalität um ClearRTTimeout
 
-\date 09.08.2006 \li V1.0.6.3 SteT
-                 \li SteT EDB 26406: Erweiterung der massaiCORBA Funktionalität um ClearRTTimeout
 
+\version 1.0.6.2 SteT
+\li 09.08.2006
+\li RoundTrip Timeout wird jetzt initial ORB weit gesetzt
 
-\date 09.08.2006 \li V1.0.6.2 SteT
-                 \li RoundTrip Timeout wird jetzt initial ORB weit gesetzt
+\version 1.0.6.1 BrN
+\li 04.07.2006
+\li TAO 1.4 mit Connection Timeout
 
-\date 04.07.2006 \li V1.0.6.1 BrN
-                 \li TAO 1.4 mit Connection Timeout
-
-\date 08.05.2006 \li V1.0.6.0 MaM
-                 \li TAO 1.4 mit RoundTrip Timeout
+\version 1.0.6.0 MaM
+\li 08.05.2006
+\li TAO 1.4 mit RoundTrip Timeout
 
 */
-
-// #define FILEDESCRIPTION  "FileDescription"  ,"Massai CORBA library\0"
-// #define INTERNALNAME     "InternalName"     ,MassaiCORBA"\0"
-// #define LEGALTRADEMARKS  "LegalTrademarks"  ,"\0"
-// #define ORIGINALFILENAME "OriginalFilename" ,"mCorba.lib\0"
-
diff --git a/SelfServiceCommon/Massai/cpp/MassaiCorba_VC6/inc/localclnt_def.h b/SelfServiceCommon/Massai/cpp/MassaiCorba_VC6/inc/localclnt_def.h
index aace85f6..2fcdb867 100644
--- a/SelfServiceCommon/Massai/cpp/MassaiCorba_VC6/inc/localclnt_def.h
+++ b/SelfServiceCommon/Massai/cpp/MassaiCorba_VC6/inc/localclnt_def.h
@@ -1,9 +1,22 @@
 #define FILE_VER_BIN     1,0,0,0
-#define FILE_VER_STR     "FileVersion", "1.0.0.0\0"
+#define FILE_VER_STR     "FileVersion", "1.0.0.0"
 
-#define FILEDESCRIPTION  "FileDescription"  ,"Corba Testclient\0"
-#define INTERNALNAME     "InternalName"     ,"Testclient\0"
-#define LEGALTRADEMARKS  "LegalTrademarks"  ,"\0"
-#define ORIGINALFILENAME "OriginalFilename" ,"testclnt.exe\0"
+/*!
+\page testclnt.exe Release Notes - testclnt.exe - test client for mcorba.lib
+
+@version 1.0.0.0
+
+\section changehistory Change history
+
+testclnt.exe
+
+\version 1.0.0.0
+\li initial version
+*/
+
+#define FILEDESCRIPTION  "FileDescription",  "Corba Testclient"
+#define INTERNALNAME     "InternalName",     "testclnt.exe"
+#define LEGALTRADEMARKS  "LegalTrademarks",  ""
+#define ORIGINALFILENAME "OriginalFilename", "testclnt.exe"
 
 
diff --git a/SelfServiceCommon/Massai/cpp/MassaiCorba_VC6/inc/localsrv_def.h b/SelfServiceCommon/Massai/cpp/MassaiCorba_VC6/inc/localsrv_def.h
index 8b75f4c0..c5f81ef6 100644
--- a/SelfServiceCommon/Massai/cpp/MassaiCorba_VC6/inc/localsrv_def.h
+++ b/SelfServiceCommon/Massai/cpp/MassaiCorba_VC6/inc/localsrv_def.h
@@ -1,9 +1,22 @@
 #define FILE_VER_BIN     1,0,0,0
-#define FILE_VER_STR     "FileVersion", "1.0.0.0\0"
+#define FILE_VER_STR     "FileVersion", "1.0.0.0"
 
-#define FILEDESCRIPTION  "FileDescription"  ,"Corba Test Servant\0"
-#define INTERNALNAME     "InternalName"     ,"Testservant\0"
-#define LEGALTRADEMARKS  "LegalTrademarks"  ,"\0"
-#define ORIGINALFILENAME "OriginalFilename" ,"testsrvr.exe\0"
+/*!
+\page testsrvr.exe Release Notes - testsrvr.exe - test servant for mcorba.lib
+
+@version 1.0.0.0
+
+\section changehistory Change history
+
+testsrvr.exe
+
+\version 1.0.0.0
+\li initial version
+*/
+
+#define FILEDESCRIPTION  "FileDescription",  "Corba Test Servant"
+#define INTERNALNAME     "InternalName",     "testsrvr.exe"
+#define LEGALTRADEMARKS  "LegalTrademarks",  ""
+#define ORIGINALFILENAME "OriginalFilename", "testsrvr.exe"
 
 
diff --git a/SelfServiceCommon/Massai/cpp/MassaiCorba_VC6/inc/massaiCORBA_def.h b/SelfServiceCommon/Massai/cpp/MassaiCorba_VC6/inc/massaiCORBA_def.h
index d233c538..9cc89f98 100644
--- a/SelfServiceCommon/Massai/cpp/MassaiCorba_VC6/inc/massaiCORBA_def.h
+++ b/SelfServiceCommon/Massai/cpp/MassaiCorba_VC6/inc/massaiCORBA_def.h
@@ -1,108 +1,102 @@
-/**
- *  @file      massaiCORBA_def.h
- *
- *  @brief     Version file
- *
- *  @date      09.05.2006
- *
- *  @author    Materna Information & Communications (MaM)
- *
- ************************************************************************/
-
-/*! \page massaicorbalib Massai CORBA library - Release Notes
- *
- *  @version 1.0.6.17
- *
- *  \section change1 Change history
- *
- *  \date 09.06.2011 \li V1.0.6.17
- *                   \li GiT, rollback of previous change
- *
- *  \date 09.06.2011 \li V1.0.6.16
- *                   \li GiT, modification: changed orb connection handler behaviour to RW
- *                            TAO description : RW selects a strategy that simply blocks in recv() when waiting for a response from the server
- *                                              instead of waiting in the Reactor using the Leader/Followers pattern.
- *                            This change should prevent nested upcalls.
- *                       To change the behaviour back to the old strategy set the environment variable
- *                       STANDARD_ORB_WAIT_STRATEGY
- *                   \li QA-Tests: All platform modules including the connection to applications are affected, a thorough integration test is needed
- *
- *  \date 21.04.2011 \li V1.0.6.15
- *                   \li SteT, enhancement, added trace to corbaorb
- *
- *  \date 28.10.2009 \li V1.0.6.14
- *                   \li SteT Mantis 11665: deactivate CorbaPoll (per default)
- *
- *     per default the corbaClient is created with the parameter AUTO_POLL_OFF
- *     this behaviour can be changed with environment setting:
- *        use CORBA_AUTOPOLL_ENABLE to enabling the polling for non local connections
- *
- *  \date 28.04.2009 \li V1.0.6.13
- *                   \li SteT bugfix: use killClientOrb when starting new Servant to avoid LOCATION_FORWARD from servant port to client port
- *
- *  \date 04.12.2008 \li V1.0.6.12
- *                   \li SteT Mantis 6871: correct CorbaPoll Tracelevel
- *
- *  \date 04.09.2008 \li V1.0.6.11
- *                   \li SteT EDB 28111: Invalid Pointer Read / Free Memory Read in CorbaPoll
- *
- *
- *  \date 17.07.2008 \li V1.0.6.10
- *                   \li SteT Change: corbaClient sends KeepAlive requests
- *
- *   corbaClient is derived from corbaPoll and the default behaviour of corbaClient ist changed:
- *
- *     per default the corbaClient is created with the parameter AUTO_POLL_NON_LOCAL
- *     if the servant is not obvious on the same machine, the client send every minute a LocateRequest message to the servant
- *
- *     the behaviour can be changed with the autopoll parameter in the constructor
- *     and with environment setting:
- *        use CORBA_AUTOPOLL_DISABLE to disable the polling
- *        use CORBA_AUTOPOLL_TIMEOUT to set the polling timeout (in ms)
- *
- *   a new corbaClient constructor is added for callback-listener set by remote (the KeepAlive mechanism is not used if the listener is used without the corbaClient)
- *
- *
- *  \date 12.11.2007 \li V1.0.6.9
- *                   \li SteT Bugfix unresolved external of CfgXMgrB (ACE_NO_INLINE removed)
- *
- *  \date 25.07.2007 \li V1.0.6.8
- *                   \li SteT EDB 26866: the classes corbaClient, corbaServantEx stores the ior and releases the memory used
- *                   \li     the method ior() returns const char* (was char*)
- *                   \li     the caller is not responsible for releasing the returned buffer any more
- *
- *  \date 20.06.2007 \li V1.0.6.7
- *                   \li SteT EDB 27607: wrong constructor results in access violation
- *
- *  \date 20.06.2007 \li V1.0.6.6
- *                   \li SteT EDB 27562: Pure virtual function call, after removeServant
- *
- *  \date 30.05.2007 \li V1.0.6.5
- *                   \li feature - hostname_in_ior can be set from external. PtM
- *
- *  \date 17.01.2007 \li V1.0.6.4 MaM
- *                   \li MaM EDB 27073: mCorba / mDisplay: kein CORBA Roundtrip TO bei Fremdplattform
- *
- *  \date 09.08.2006 \li V1.0.6.3 SteT
- *                   \li SteT EDB 26406: Erweiterung der massaiCORBA Funktionalität um ClearRTTimeout
- *
- *
- *  \date 09.08.2006 \li V1.0.6.2 SteT
- *                   \li RoundTrip Timeout wird jetzt initial ORB weit gesetzt
- *
- *  \date 04.07.2006 \li V1.0.6.1 BrN
- *                   \li TAO 1.4 mit Connection Timeout
- *
- *  \date 08.05.2006 \li V1.0.6.0 MaM
- *                   \li TAO 1.4 mit RoundTrip Timeout
- *
- */
-
-// #define FILE_VER_BIN     1,0,6,17
-// #define FILE_VER_STR     "FileVersion", "1.0.6.17\0"
-
-// #define FILEDESCRIPTION  "FileDescription"  ,"Massai CORBA library\0"
-// #define INTERNALNAME     "InternalName"     ,MassaiCORBA"\0"
-// #define LEGALTRADEMARKS  "LegalTrademarks"  ,"\0"
-// #define ORIGINALFILENAME "OriginalFilename" ,"mCorba.lib\0"
+/*!
+\page mcorba.lib_VC6 Release Notes - mcorba.lib - Massai CORBA library
 
+@version 1.0.6.17
+
+\section changehistory Change history
+
+mcorba.lib
+
+\version 1.0.6.17
+\li 09.06.2011
+\li GiT, rollback of previous change
+
+\version 1.0.6.16
+\li 09.06.2011
+\li GiT, modification: changed orb connection handler behaviour to RW\n
+    TAO description : RW selects a strategy that simply blocks in recv() when waiting for a response from the server\n
+                      instead of waiting in the Reactor using the Leader/Followers pattern.\n
+    This change should prevent nested upcalls.\n
+    To change the behaviour back to the old strategy set the environment variable\n
+    STANDARD_ORB_WAIT_STRATEGY
+\li QA-Tests: All platform modules including the connection to applications are affected, a thorough integration test is needed
+
+\version 1.0.6.15
+\li 21.04.2011
+\li SteT, enhancement, added trace to corbaorb
+
+\version 1.0.6.14
+\li 28.10.2009
+\li SteT Mantis 11665: deactivate CorbaPoll (per default)\n
+   per default the corbaClient is created with the parameter AUTO_POLL_OFF\n
+   this behaviour can be changed with environment setting:\n
+      use CORBA_AUTOPOLL_ENABLE to enabling the polling for non local connections
+
+\version 1.0.6.13
+\li 28.04.2009
+\li SteT bugfix: use killClientOrb when starting new Servant to avoid LOCATION_FORWARD from servant port to client port
+
+\version 1.0.6.12
+\li 04.12.2008
+\li SteT Mantis 6871: correct CorbaPoll Tracelevel
+
+\version 1.0.6.11
+\li 04.09.2008
+\li SteT EDB 28111: Invalid Pointer Read / Free Memory Read in CorbaPoll
+
+\version 1.0.6.10
+\li 17.07.2008
+\li SteT Change: corbaClient sends KeepAlive requests
+\li corbaClient is derived from corbaPoll and the default behaviour of corbaClient ist changed:\n
+   per default the corbaClient is created with the parameter AUTO_POLL_NON_LOCAL\n
+   if the servant is not obvious on the same machine, the client send every minute a LocateRequest message to the servant\n
+   the behaviour can be changed with the autopoll parameter in the constructor\n
+   and with environment setting:\n
+      use CORBA_AUTOPOLL_DISABLE to disable the polling\n
+      use CORBA_AUTOPOLL_TIMEOUT to set the polling timeout (in ms)
+\li a new corbaClient constructor is added for callback-listener set by remote (the KeepAlive mechanism is not used if the listener is used without the corbaClient)
+
+\version 1.0.6.9
+\li 12.11.2007
+\li SteT Bugfix unresolved external of CfgXMgrB (ACE_NO_INLINE removed)
+
+\version 1.0.6.8
+\li 25.07.2007
+\li SteT EDB 26866: the classes corbaClient, corbaServantEx stores the ior and releases the memory used
+\li     the method ior() returns const char* (was char*)
+\li     the caller is not responsible for releasing the returned buffer any more
+
+\version 1.0.6.7
+\li 20.06.2007
+\li SteT EDB 27607: wrong constructor results in access violation
+
+\version 1.0.6.6
+\li 20.06.2007
+\li SteT EDB 27562: Pure virtual function call, after removeServant
+
+\version 1.0.6.5
+\li 30.05.2007
+\li feature - hostname_in_ior can be set from external. PtM
+
+\version 1.0.6.4 MaM
+\li 17.01.2007
+\li MaM EDB 27073: mCorba / mDisplay: kein CORBA Roundtrip TO bei Fremdplattform
+
+\version 1.0.6.3 SteT
+\li 09.08.2006
+\li SteT EDB 26406: Erweiterung der massaiCORBA Funktionalität um ClearRTTimeout
+
+
+\version 1.0.6.2 SteT
+\li 09.08.2006
+\li RoundTrip Timeout wird jetzt initial ORB weit gesetzt
+
+\version 1.0.6.1 BrN
+\li 04.07.2006
+\li TAO 1.4 mit Connection Timeout
+
+\version 1.0.6.0 MaM
+\li 08.05.2006
+\li TAO 1.4 mit RoundTrip Timeout
+
+*/
diff --git a/SelfServiceCommon/Massai/cpp/StatisticsService/inc/StatisticsClnt_def.h b/SelfServiceCommon/Massai/cpp/StatisticsService/inc/StatisticsClnt_def.h
index 2977057d..608147f9 100644
--- a/SelfServiceCommon/Massai/cpp/StatisticsService/inc/StatisticsClnt_def.h
+++ b/SelfServiceCommon/Massai/cpp/StatisticsService/inc/StatisticsClnt_def.h
@@ -1,9 +1,22 @@
 #define FILE_VER_BIN     1,0,0,1
-#define FILE_VER_STR     "FileVersion", "1.0.0.1\0"
+#define FILE_VER_STR     "FileVersion", "1.0.0.1"
 
-#define FILEDESCRIPTION  "FileDescription"  ,"Statistics client\0"
-#define INTERNALNAME     "InternalName"     ,"Statistics client\0"
-#define LEGALTRADEMARKS  "LegalTrademarks"  ,"\0"
-#define ORIGINALFILENAME "OriginalFilename" ,"StatisticsClnt.exe\0"
+/*!
+\page StatisticsClnt.exe Release Notes - StatisticsClnt.exe - test client for mcorba.lib
+
+@version 1.0.0.1
+
+\section changehistory Change history
+
+StatisticsClnt.exe
+
+\version 1.0.0.1
+\li initial version
+*/
+
+#define FILEDESCRIPTION  "FileDescription",  "Statistics client"
+#define INTERNALNAME     "InternalName",     "StatisticsClnt.exe"
+#define LEGALTRADEMARKS  "LegalTrademarks",  ""
+#define ORIGINALFILENAME "OriginalFilename", "StatisticsClnt.exe"
 
 
diff --git a/SelfServiceCommon/Massai/cpp/StatisticsService/inc/Statistics_def.h b/SelfServiceCommon/Massai/cpp/StatisticsService/inc/Statistics_def.h
index dd9f8923..5548c716 100644
--- a/SelfServiceCommon/Massai/cpp/StatisticsService/inc/Statistics_def.h
+++ b/SelfServiceCommon/Massai/cpp/StatisticsService/inc/Statistics_def.h
@@ -1,83 +1,90 @@
 #define FILE_VER_BIN     1,0,1,4
-#define FILE_VER_STR     "FileVersion", "1.0.1.4\0"
+#define FILE_VER_STR     "FileVersion", "1.0.1.4"
 
 /*!
-\page Statistics.exe Statistics Service - Release Notes
+\page Statistics.exe Release Notes - Statistics.exe - statistics service
 
-\date 07.11.2012 \li V1.0.1.4
-                 \li SteT Mantis 0024999: service creates very large files (VC10)
+@version 1.0.1.4
 
-\date 16.12.2011 \li V1.0.1.3
-                 \li SteT Mantis 0021947: mstarter: problems at shutdown
-                     shutdown on SIGBREAK, ignore CTRL_LOGOFF_EVENT
+\section changehistory Change history
 
-\date 17.11.2011 \li V 1.0.1.2
-                 \li GiT: Mantis0021568: SelfServiceCommon StatisticsService changes for VC10 compatibilty
+statistics.exe
 
-\date 12.04.2011 \li V 1.0.1.1
-                 \li PtM: convert '"' -> '&quot;' to avoid values parsing problems
+\version 1.0.1.4
+\li 07.11.2012
+\li SteT Mantis 0024999: service creates very large files (VC10)
 
-\date 18.06.2008 \li V 1.0.1.0
-                 \li PtM: new Statistics constructor. (more data in the header)
+\version 1.0.1.3
+\li 16.12.2011
+\li SteT Mantis 0021947: mstarter: problems at shutdown
+\li shutdown on SIGBREAK, ignore CTRL_LOGOFF_EVENT
 
-\date 14.05.2008 \li V 1.0.0.12
-                 \li SteT EDB 28033: define name/partitioning of statistics files with configuration
+\version  1.0.1.2
+\li 17.11.2011
+\li GiT: Mantis0021568: SelfServiceCommon StatisticsService changes for VC10 compatibilty
 
-\li new configuration item SERVICES.STATISTICS.OutputFilename added
+\version  1.0.1.1
+\li 12.04.2011
+\li PtM: convert '"' -> '&quot;' to avoid values parsing problems
 
-   the pattern %KEY(:LENGTH)?%
-                   with KEY=[A-Za-z0-9_]+
-                        LENGTH=[0-9]+
-
-   is replaced by the according value (trimmed to the length if defined)
-
-   the following keys are supported
-       KIOSKID
-       LOCATION
-       APPNAME
-       APPTYPE
-       DAYOFYEAR                // 001..366
-       DAYOFMONTH               // 01..31
-       MONTHOFYEAR              // 01..12 (01: january, ...)
-       DAYOFWEEK                // 1..6 (1: monday, ...)
-       WEEKOFYEAR               // 01..53
-       WEEKOFMONTH              // 1..5
-       MONTHFORWEEK             // 01..12 (1: january, ...)
-       YEARFORWEEK              // 1970...
-       YEAR                     // 1970...
-       YEARFORWEEK_IC           // 00...99
-       YEAR_IC                  // 00...99
-       HOUR                     // 00..23
-       MINUTE                   // 00..59
-       SECOND                   // 00..59,60,61
-
-   all other keys are just removed from the text
-
-   the default value Stat%YEAR_IC%%MONTHOFYEAR%.xml produces the same filename as the previous version
+\version  1.0.1.0
+\li 18.06.2008
+\li PtM: new Statistics constructor. (more data in the header)
 
+\version  1.0.0.12
+\li 14.05.2008
+\li SteT EDB 28033: define name/partitioning of statistics files with configuration
+\li new configuration item SERVICES.STATISTICS.OutputFilename added
+\li the pattern %KEY(:LENGTH)?%\n
+                   with KEY=[A-Za-z0-9_]+\n
+                        LENGTH=[0-9]+\n
+   is replaced by the according value (trimmed to the length if defined)\n
+   the following keys are supported\n
+       KIOSKID\n
+       LOCATION\n
+       APPNAME\n
+       APPTYPE\n
+       DAYOFYEAR                // 001..366\n
+       DAYOFMONTH               // 01..31\n
+       MONTHOFYEAR              // 01..12 (01: january, ...)\n
+       DAYOFWEEK                // 1..6 (1: monday, ...)\n
+       WEEKOFYEAR               // 01..53\n
+       WEEKOFMONTH              // 1..5\n
+       MONTHFORWEEK             // 01..12 (1: january, ...)\n
+       YEARFORWEEK              // 1970...\n
+       YEAR                     // 1970...\n
+       YEARFORWEEK_IC           // 00...99\n
+       YEAR_IC                  // 00...99\n
+       HOUR                     // 00..23\n
+       MINUTE                   // 00..59\n
+       SECOND                   // 00..59,60,61\n
+   all other keys are just removed from the text\n
+   the default value Stat%YEAR_IC%%MONTHOFYEAR%.xml produces the same filename as the previous version\n
 \li Statistics.exe needs vc6-re300l.dll
-
 \li the keys are also available for the configuration item SERVICES.STATISTICS.OutputPath
 
-\date 06.09.2007 \li V 1.0.0.11
-                 \li SteT EDB 27806: write access to log file synchronized
-
-\date 24.08.2007 \li V 1.0.0.10
-                 \li bugfix: reduced memory leak caused by config access (PtM).
+\version  1.0.0.11
+\li 06.09.2007
+\li SteT EDB 27806: write access to log file synchronized
 
-\date 27.04.2006 \li V 1.0.0.9
-                 \li SteT EDB 26729: Remove CTRL_BREAK
+\version  1.0.0.10
+\li 24.08.2007
+\li bugfix: reduced memory leak caused by config access (PtM).
 
-\date 25.10.2006 \li V1.0.0.8
-                 \li SteT EDB 26693: Abgleich des neuen CC Streams SelfServicePlatform mit den alten Streams
-                 \li limit trace output
+\version  1.0.0.9
+\li 27.04.2006
+\li SteT EDB 26729: Remove CTRL_BREAK
 
+\version 1.0.0.8
+\li 25.10.2006
+\li SteT EDB 26693: Abgleich des neuen CC Streams SelfServicePlatform mit den alten Streams
+\li limit trace output
  */
 
-#define FILEDESCRIPTION  "FileDescription"  ,"Statistics Service\0"
-#define INTERNALNAME     "InternalName"     ,"Statistics\0"
-#define LEGALTRADEMARKS  "LegalTrademarks"  ,"\0"
-#define ORIGINALFILENAME "OriginalFilename" ,"Statistics.exe\0"
+#define FILEDESCRIPTION  "FileDescription",  "Statistics Service"
+#define INTERNALNAME     "InternalName",     "Statistics.exe"
+#define LEGALTRADEMARKS  "LegalTrademarks",  ""
+#define ORIGINALFILENAME "OriginalFilename", "Statistics.exe"
 
 
 
diff --git a/SelfServiceCommon/Massai/cpp/Systools/src/local_def.cpp b/SelfServiceCommon/Massai/cpp/Systools/src/local_def.cpp
index 4195541e..8600a44f 100644
--- a/SelfServiceCommon/Massai/cpp/Systools/src/local_def.cpp
+++ b/SelfServiceCommon/Massai/cpp/Systools/src/local_def.cpp
@@ -1,105 +1,101 @@
-#include "Tools/VERSIONLIB.h"
+#include "tools/versionlib.h"
 
 VERSION_ID_DT(systools, 1, 0, 0, 13);
 
 /*!
-\page massai_systools.lib, massai_systools.lib - Release Notes
+\page massai_systools.lib Release Notes - massai_systools.lib
+
+@version 1.0.0.3
+
+\section changehistory Change history
 
 massai_systools.lib
 
-\date 25.09.2013
-\li V1.0.0.13
+\version 1.0.0.13
+\li 25.09.2013
 \li Mantis 0028892: PLF: SAM-Hotkey not working
-mSysHotKey: listen on windows hot keys
-mSysThreadPool: process jobs in a pool of threads
+\li mSysHotKey: listen on windows hot keys
+\li mSysThreadPool: process jobs in a pool of threads
 
-\date 19.09.2013
-\li V1.0.0.12
+\version 1.0.0.12
+\li 19.09.2013
 \li Mantis 0028903: MR: MCR Sankyo ICM 350
-mSysDeviceEvent: device name stops at terminating zero
-mSysIO: open files with read and write share
+\li mSysDeviceEvent: device name stops at terminating zero
+\li mSysIO: open files with read and write share\n
         do not dump input (PCI DSS)
 
 
-\date 17.09.2013
-\li V1.0.0.11
+\version 1.0.0.11
+\li 17.09.2013
 \li Mantis 0029028: mstarter: if process cannot be started, the pid 0 is used to find subprocesses
-  ignore subprocesses if a processes cannot be started
+\li   ignore subprocesses if a processes cannot be started
 
-\date 16.09.2013
-\li V1.0.0.10
+\version 1.0.0.10
+\li 16.09.2013
 \li Mantis 0028903: MR: MCR Sankyo ICM 350
-  added SystemError for exceptions in mSysErrorW32.h
-  added mSysUSBHID.h for finding hid devices
-  added DeviceSearch for finding all or only present USB devices
-  moved common implementation from mSysUSBPrinter to mSysUSBDevice
+\li   added SystemError for exceptions in mSysErrorW32.h
+\li   added mSysUSBHID.h for finding hid devices
+\li   added DeviceSearch for finding all or only present USB devices
+\li   moved common implementation from mSysUSBPrinter to mSysUSBDevice
 
-\date 04.09.2013
-\li V1.0.0.9
+\version 1.0.0.9
+\li 04.09.2013
 \li Mantis 0028873: GPPDev: Custom KPM150H RFID
-added mSysIO (timed io)
-added mSysUSBPrinter (find USB device by printer port name)
-added functions for calculation in mSysTime
+\li added mSysIO (timed io)
+\li added mSysUSBPrinter (find USB device by printer port name)
+\li added functions for calculation in mSysTime
 
-\date 15.08.2013
-\li V1.0.0.8
+\version 1.0.0.8
+\li 15.08.2013
 \li Mantis 0028547: mstarter: crashes sometimes at platform start
-race condition at termination of processes:
+\li race condition at termination of processes:\n
   ProcessList deletes ProcessStart deletes ProcessHolder while ProcessHolder handles termination notification
 
 
-\date 31.07.2013
-\li V1.0.0.7
+\version 1.0.0.7
+\li 31.07.2013
 \li Mantis 0028323: GPPDev: Custom VKP-80II
-added mSysDeviceEvent; listen on changes in device configuration (USB device connected; disconnected)
-added mSysEvent; listen on system events; helper for mSysDeviceEvent
+\li added mSysDeviceEvent; listen on changes in device configuration (USB device connected; disconnected)
+\li added mSysEvent; listen on system events; helper for mSysDeviceEvent
 
 
-\date 09.07.2013
-\li V1.0.0.6
+\version 1.0.0.6
+\li 09.07.2013
 \li Mantis 0027919: appControl: desktop can not be closed; ctfmon.exe must be terminated;
-mSysProcess::child added method to get parent process id: std::string getParentProcessId() const;
-mSysProcessTree::ProcessNode added method to get parent process id: std::string getParentProcessId() const;
-mSysWindow::Window added method to get owner process id: std::string getProcessId();
-                   added method to check if the window is closed: bool isClosed();
-
-mSysProcessList::ProcessProperties added list of window ids for termination
+\li mSysProcess::child added method to get parent process id: std::string getParentProcessId() const;
+\li mSysProcessTree::ProcessNode added method to get parent process id: std::string getParentProcessId() const;
+\li mSysWindow::Window added method to get owner process id: std::string getProcessId();\n
+                       added method to check if the window is closed: bool isClosed();
+\li mSysProcessList::ProcessProperties added list of window ids for termination\n
  use CLOSE_WND = "title" CLOSE_WNDCLASS = "class" in option list
 
-
-
-\date 27.06.2013
-\li V1.0.0.5
+\version 1.0.0.5
+\li 27.06.2013
 \li Mantis 0026902: Internet Explorer Process doesn't close
-
-ApplicationList, ProcessList closes child processes; except processes of an exlusion list; mSysProcessList.h
-add ProcessTree for finding child processes; mSysProcessTree.h
-add compare operators for Timestamp, Duration; mSysTime.h
-mSysProcess::child provide program name and creation time; mSysProcess.hpp
-
-\date 05.02.2013 \li V1.0.0.4
-                 \li Mantis 0025897: GPPDev: NCR K8 workaround for crash of printer driver
-                     add serviceMonitor to systools (monitor status of windows services)
-
-\date 27.11.2012 \li V1.0.0.3
-                 \li SteT Mantis 0024715: Build: VC10; fix compiler warnings
-
-\date 18.04.2012 \li V1.0.0.2
-                 \li SteT Mantis 0023016: msystools: msysthread invalid handle
-
-\date 03.11.2011 \li V1.0.0.1
-                 \li SteT added
-  added files mAutoMutexCS.h, mMutexCS.h, mMutexCSW32.cpp
-  with classes AutoMutexCS, Mutex
-     using CRITICAL_SECTION as more efficent synchronisation mechanism
-
-  can be use instead of
-    the classes autoMutex, mutexSemaphore
-    from the files mAutoMutexCS.h, mMutexSem.hpp, mMutexSemW32.cpp
-
-
-\date 19.10.2010 \li V1.0.0.0
-                 \li SteT Initial Version
-
+\li ApplicationList, ProcessList closes child processes; except processes of an exlusion list; mSysProcessList.h
+\li add ProcessTree for finding child processes; mSysProcessTree.h
+\li add compare operators for Timestamp, Duration; mSysTime.h
+\li mSysProcess::child provide program name and creation time; mSysProcess.hpp
+
+\li 05.02.2013 \version 1.0.0.4
+\li Mantis 0025897: GPPDev: NCR K8 workaround for crash of printer driver
+\li add serviceMonitor to systools (monitor status of windows services)
+
+\li 27.11.2012 \version 1.0.0.3
+\li SteT Mantis 0024715: Build: VC10; fix compiler warnings
+
+\li 18.04.2012 \version 1.0.0.2
+\li SteT Mantis 0023016: msystools: msysthread invalid handle
+
+\li 03.11.2011 \version 1.0.0.1
+\li SteT added
+\li added files mAutoMutexCS.h, mMutexCS.h, mMutexCSW32.cpp\n
+  with classes AutoMutexCS, Mutex\n
+     using CRITICAL_SECTION as more efficent synchronisation mechanism\n
+  can be use instead of\n
+    the classes autoMutex, mutexSemaphore\n
+    from the files mAutoMutexCS.h, mMutexSem.hpp, mMutexSemW32.cpp\n
+
+\li 19.10.2010 \version 1.0.0.0
+\li SteT Initial Version
 */
-
diff --git a/SelfServiceCommon/Massai/cpp/Text/src/local_def.cpp b/SelfServiceCommon/Massai/cpp/Text/src/local_def.cpp
index 49688b14..91a7875f 100644
--- a/SelfServiceCommon/Massai/cpp/Text/src/local_def.cpp
+++ b/SelfServiceCommon/Massai/cpp/Text/src/local_def.cpp
@@ -1,40 +1,44 @@
-#include "Tools/VERSIONLIB.h"
+#include "tools/versionlib.h"
 
 VERSION_ID_DT(text, 1, 0, 0, 5);
 
-/*! \page texttools.lib; text tools library - Release Notes
+/*! \page massai_text.lib Release Notes - massai_text.lib - text tool library
+
+@version 1.0.0.3
+
+\section changehistory Change history
 
 massai_text.lib
 
-\date 16.09.2013
-\li V1.0.0.5
+\version 1.0.0.5
+\li 16.09.2013
 \li Mantis 0028903: MR: MCR Sankyo ICM 350
   added replaceAll
 
-\date 09.07.2013
-\li V1.0.0.4
+\version 1.0.0.4
+\li 09.07.2013
 \li Mantis 0027919: appControl: desktop can not be closed; ctfmon.exe must be terminated;
   remove default: TerminationExclusion=ctfmon.exe from PLATFORM.ENVIRONMENT.Variables
 
-\date 27.06.2013
-\li V1.0.0.3
+\version 1.0.0.3
+\li 27.06.2013
 \li SteT Mantis 0027839: mstarter: substition for "$"
 ignore environment variables with empty key
 
-\date 27.06.2013
-\li V1.0.0.2
+\version 1.0.0.2
+\li 27.06.2013
 \li SteT Mantis 0026902: Internet Explorer Process doesn't close
-readEnv added
-  read environment variables
-  read configuration entry
-    PLATFORM.ENVIRONMENT.Variables
-    type: text list
-    description: substitution variables; format variablename=value
-    default: TerminationExclusion=ctfmon.exe
-
-\date 19.11.2010 \li V1.0.0.1
-                 \li SteT textTools added
-
-\date 19.10.2010 \li V1.0.0.0
-                 \li SteT Initial Version
+\li readEnv added\n
+  read environment variables\n
+  read configuration entry\n
+    PLATFORM.ENVIRONMENT.Variables\n
+    type: text list\n
+    description: substitution variables; format variablename=value\n
+    default: TerminationExclusion=ctfmon.exe\n
+
+\li 19.11.2010 \version 1.0.0.1
+\li SteT textTools added
+
+\li 19.10.2010 \version 1.0.0.0
+\li SteT Initial Version
 */
diff --git a/SelfServiceCommon/Massai/cpp/Tools/inc/local_def_msystools.h b/SelfServiceCommon/Massai/cpp/Tools/inc/local_def_msystools.h
index 585ab189..fd05d092 100644
--- a/SelfServiceCommon/Massai/cpp/Tools/inc/local_def_msystools.h
+++ b/SelfServiceCommon/Massai/cpp/Tools/inc/local_def_msystools.h
@@ -1,98 +1,111 @@
 #define FILE_VER_BIN     1,0,1,13
-#define FILE_VER_STR     "FileVersion", "1.0.1.13\0"
+#define FILE_VER_STR     "FileVersion", "1.0.1.13"
 
 /*!
-\page tools.lib tools.lib - Release Notes
+\page massai_tools.lib Release Notes - massai_tools.lib
+
+@version 1.0.1.13
+
+\section changehistory Change history
 
 tools.lib
 
-\date 07.08.2013
-\li V1.0.1.13
+\version 1.0.1.13
+\li 07.08.2013
 \li SteT - Mantis 0028323: GPPDev: Custom VKP-80II
 \li extension: added Interval.h
 
-
-\date 06.08.2013
-\li V1.0.1.12
+\version 1.0.1.12
+\li 06.08.2013
 \li SteT - Mantis 0028323: GPPDev: Custom VKP-80II
 \li extension: added tracesToTraceInterface.h
 
-
-\date 31.08.2012 \li V1.0.1.11
-                 \li SteT -  Mantis 0020322: hs882 send a lot of data_present
-                 \li extension: added ActionTimeLimit: restrict the number of actions per period
-
-\date 15.06.2012 \li V1.0.1.10
-                 \li SteT -  Mantis  0023249: GPP: support two different papertypes for a single printer
-                 \li bugfix: resize from size 0 to an other size does not reserve memory
-
-\date 19.03.2012 \li V1.0.1.9
-                 \li SteT -  Mantis 0017128: TrcServ: replace communication layer
-                     added TraceSetLevel and TraceSetLevelDelegate to filter trace by tracelevel/topic
-
-\date 21.10.2010 \li V1.0.1.8
-                 \li SteT -  Mantis 10547: TrcServ: trace client functions return type: void
-
-
-\date 22.09.2010 \li V1.0.1.7
-                 \li SteT - Added
-                 \li   - interface barcode, implementation barcode39, barcode128, barcodeIndustrial2of5
-                 \li            create barcode bitmap
-                 \li   - interface bitmap, implementation blackAndWhiteBitmap
-                 \li            create bitmap from stringbuffer
-                 \li   - bitarray
-                 \li            access single bit
-                 \li   - pixelrow
-                 \li            create a bitmap of a single bitarray (repeating the line)
-                 \li   - stringbuffer
-                 \li            hold a char* buffer, allow direct access to char* (read and write)
-                 \li   - localtrace
-                 \li            write traces to stdout
-
-\date 23.06.2009 \li V1.0.1.6
-                 \li JK - Added mSysThread module.
-                 \li      This module replaces matThread. Differences:
-                 \li      - Restarting a thread fixed.
-                 \li      - the start() method no longer waits for the
-                 \li        thread function to run.
-                 \li      - Thread function exceptions are caught. They
-                 \li        will no longer cause program termination.
-
-\date 05.02.2009 \li V1.0.1.5
-                 \li JK - Extended mSysProcess module.
-                 \li      - allow also to redirect stdin
-                 \li      - allow to hide a child window
-
-\date 04.02.2009 \li V1.0.1.4
-                 \li JK - Added mSysProcess module.
-                 \li      It allows to spawn child processes with re-directed stdout/stderr and
-                 \li      to control process status and exit code.
-
-\date 13.01.2009 \li V1.0.1.3
-                 \li JK - Added argScan implementation in mArgScan.cpp.
-                 \li      Previously implemented inline in header file.
-                 \li    - New function tellIgnored()
-                 \li      to check for unused/superflous parameters
-
-\date 02.12.2008 \li V1.0.1.2
-                 \li SteT - bugfix main exported from matDaemonW32.cpp renamed
-                 \li        to mDaemon_execute (mDaemon_main already in use)
-
-\date 14.05.2008 \li V1.0.1.1
-                 \li SteT - added dateProperties, calculates weekOfYear, weekOfMonth, ...
+\version 1.0.1.11
+\li 31.08.2012
+\li SteT -  Mantis 0020322: hs882 send a lot of data_present
+\li extension: added ActionTimeLimit: restrict the number of actions per period
+
+\version 1.0.1.10
+\li 15.06.2012
+\li SteT -  Mantis  0023249: GPP: support two different papertypes for a single printer
+\li bugfix: resize from size 0 to an other size does not reserve memory
+
+\version 1.0.1.9
+\li 19.03.2012
+\li SteT -  Mantis 0017128: TrcServ: replace communication layer
+\li added TraceSetLevel and TraceSetLevelDelegate to filter trace by tracelevel/topic
+
+\version 1.0.1.8
+\li 21.10.2010
+\li SteT -  Mantis 10547: TrcServ: trace client functions return type: void
+
+\version 1.0.1.7
+\li 22.09.2010
+\li SteT - Added
+\li   - interface barcode, implementation barcode39, barcode128, barcodeIndustrial2of5
+\li            create barcode bitmap
+\li   - interface bitmap, implementation blackAndWhiteBitmap
+\li            create bitmap from stringbuffer
+\li   - bitarray
+\li            access single bit
+\li   - pixelrow
+\li            create a bitmap of a single bitarray (repeating the line)
+\li   - stringbuffer
+\li            hold a char* buffer, allow direct access to char* (read and write)
+\li   - localtrace
+\li            write traces to stdout
+
+\version 1.0.1.6
+\li 23.06.2009
+\li JK - Added mSysThread module.
+\li      This module replaces matThread. Differences:
+\li      - Restarting a thread fixed.
+\li      - the start() method no longer waits for the
+\li        thread function to run.
+\li      - Thread function exceptions are caught. They
+\li        will no longer cause program termination.
+
+\version 1.0.1.5
+\li 05.02.2009
+\li JK - Extended mSysProcess module.
+\li      - allow also to redirect stdin
+\li      - allow to hide a child window
+
+\version 1.0.1.4
+\li 04.02.2009
+\li JK - Added mSysProcess module.
+\li      It allows to spawn child processes with re-directed stdout/stderr and
+\li      to control process status and exit code.
+
+\version 1.0.1.3
+\li 13.01.2009
+\li JK - Added argScan implementation in mArgScan.cpp.
+\li      Previously implemented inline in header file.
+\li    - New function tellIgnored()
+\li      to check for unused/superflous parameters
+
+\version 1.0.1.2
+\li 02.12.2008
+\li SteT - bugfix main exported from matDaemonW32.cpp renamed
+\li        to mDaemon_execute (mDaemon_main already in use)
+
+\version 1.0.1.1
+\li 14.05.2008
+\li SteT - added dateProperties, calculates weekOfYear, weekOfMonth, ...
                             added textTemplate fill a %([A-Za-z0-9_]+)(:[0-9]+)?%-template
 
-\date 12.10.2007 \li V1.0.1.0
-                 \li AkA - added trace streamer for stream-like tracing
-
-\date 28.06.2007 \li V1.0.0.0
-                 \li SteT EDB 27099: combine tools into a single lib
+\version 1.0.1.0
+\li 12.10.2007
+\li AkA - added trace streamer for stream-like tracing
 
+\version 1.0.0.0
+\li 28.06.2007
+\li SteT EDB 27099: combine tools into a single lib
 */
 
-#define FILEDESCRIPTION  "FileDescription"  ,"tools"
-#define INTERNALNAME     "InternalName"     ,"tools\0"
-#define LEGALTRADEMARKS  "LegalTrademarks"  ,"\0"
-#define ORIGINALFILENAME "OriginalFilename" ,"tools.lib\0"
+#define FILEDESCRIPTION  "FileDescription",  "tools"
+#define INTERNALNAME     "InternalName",     "massai_tools.lib"
+#define LEGALTRADEMARKS  "LegalTrademarks",  ""
+#define ORIGINALFILENAME "OriginalFilename", "massai_tools.lib"
 
 
diff --git a/SelfServiceCommon/Massai/cpp/Trace/inc/TraceDLL_def.h b/SelfServiceCommon/Massai/cpp/Trace/inc/TraceDLL_def.h
index 0ae26f24..6a30b227 100644
--- a/SelfServiceCommon/Massai/cpp/Trace/inc/TraceDLL_def.h
+++ b/SelfServiceCommon/Massai/cpp/Trace/inc/TraceDLL_def.h
@@ -1,17 +1,9 @@
-/**
- *  @file      TraceDLL_def.h
- *
- *  @brief     Version constants
- *
- *  @date      02.11.2011
- *
- ************************************************************************/
-
+// Version constans
 #include "local_def.h"
 
-#define FILEDESCRIPTION  "FileDescription"  ,"Trace DLL\0"
-#define INTERNALNAME     "InternalName"     ,"Trace.dll\0"
-#define LEGALTRADEMARKS  "LegalTrademarks"  ,"\0"
-#define ORIGINALFILENAME "OriginalFilename" ,"Trace.dll\0"
+#define FILEDESCRIPTION  "FileDescription",  "Tracing library"
+#define INTERNALNAME     "InternalName",     "Trace.dll"
+#define ORIGINALFILENAME "OriginalFilename", "Trace.dll"
+#define LEGALTRADEMARKS  "LegalTrademarks",  ""
 
 
diff --git a/SelfServiceCommon/Massai/cpp/Trace/inc/TraceHDLDLL_def.h b/SelfServiceCommon/Massai/cpp/Trace/inc/TraceHDLDLL_def.h
index bdd946e1..fac2b211 100644
--- a/SelfServiceCommon/Massai/cpp/Trace/inc/TraceHDLDLL_def.h
+++ b/SelfServiceCommon/Massai/cpp/Trace/inc/TraceHDLDLL_def.h
@@ -1,18 +1,9 @@
-/**
- *  @file      TrcHDLDLL_def.h
- *
- *  @brief     Version constants
- *
- *  @date      02.11.2011
- *
- ************************************************************************/
-
-
+// Version constans
 #include "local_def.h"
 
-#define FILEDESCRIPTION  "FileDescription"  ,"Trace HDL\0"
-#define INTERNALNAME     "InternalName"     ,"TraceHDL.dll\0"
-#define LEGALTRADEMARKS  "LegalTrademarks"  ,"\0"
-#define ORIGINALFILENAME "OriginalFilename" ,"TraceHDL.dll\0"
+#define FILEDESCRIPTION  "FileDescription",  "Tracing library"
+#define INTERNALNAME     "InternalName",     "TraceHDL.dll"
+#define ORIGINALFILENAME "OriginalFilename", "TraceHDL.dll"
+#define LEGALTRADEMARKS  "LegalTrademarks",  ""
 
 
diff --git a/SelfServiceCommon/Massai/cpp/Trace/inc/local_def.h b/SelfServiceCommon/Massai/cpp/Trace/inc/local_def.h
index 19490f4d..82d70b16 100644
--- a/SelfServiceCommon/Massai/cpp/Trace/inc/local_def.h
+++ b/SelfServiceCommon/Massai/cpp/Trace/inc/local_def.h
@@ -1,98 +1,88 @@
 #define FILE_VER_BIN     1,0,0,5
 #define FILE_VER_STR     "FileVersion", "1.0.0.5\0"
 
+/*!
+\page trace.dl Release Notes - trace.dll, tracehdl.dll, traceNoConfig.dll, tracehdlNoConfig.dll - tracing library
+
+\section changehistory Change history
+
+@version 1.0.0.5
+
+trace.dll, tracehdl.dll, traceNoConfig.dll, tracehdlNoConfig.dll
+
+\version 1.0.0.5
+\li 23.10.2013
+\li bugfix -  Mantis 0029551: Buildsystem: Integrate Boost 1.54
+
+\version 1.0.0.4
+\li 19.10.2012
+\li bugfix -  Mantis 0024741: TraceDLL: use current working directory for relative paths
+\li bugfix -  Mantis 0024740: TraceDLL: add Interface to request TraceLevel/Topic
+                new header SelfServiceCommon\\Interfaces\\inc\\trc\\TrcHdlLevel.h
+                           SelfServiceCommon\\Interfaces\\inc\\trc\\TraceLevel.h
+
+\version 1.0.0.3
+\li 19.03.2012
+\li SteT -  Mantis 0017128: TrcServ: replace communication layer
+                     conflict with massai-tools.lib renamed TraceCallback to TraceLineCallback
+
+\version 1.0.0.2
+\li 23.02.2012
+\li SteT Mantis 0022536: trace.dll: do not add \\n to \\r; use mode binary
+
+\version 1.0.0.1
+\li 11.01.2012
+\li SteT Mantis 0022076: Trace.dll: write traces before openTrace and after closeTrace to stdout
+
+\version 1.0.0.0
+\li 27.10.2011
+\li SteT Initial Version
+ sessionId: replace '<>:'/\\|?*.' by '\_'\n
+ topic:     replace '.' by '\_'\n
+\li Settings read from environment:
+\li 'TRACE\_USE\_CONFIG': \n
+     'false'/'inactive': do not use configuration\n
+     'true'/'active':    read entries from configuration\n
+     default: 'true'\n
+\li 'TRACE\_TOPIC\_' + topic\n
+     'false'/'inactive': do not write traces for this topic\n
+     'true'/'active':    write traces for this topic\n
+     default: 'true'\n
+\li 'TRACE\_TRACELEVEL'\n
+     '1'-'9': only trace with a tracelevel lower or equal than the this value are written\n
+     default: '5'\n
+\li 'TRACE\_DIRECTORY'\n
+     The trace files are stored in this directory; relative pathes apply to the directory of the executable\n
+     default: ''\n
+\li 'TRACESESSION\_' + sessionId + '\_TOPIC\_' + topic\n
+     same as 'TRACE\_TOPIC\_' + topic\n
+     overwrites the value of 'TRACE\_TOPIC\_' + topic\n
+\li 'TRACESESSION\_' + sessionId + '\_TRACELEVEL'\n
+     same as 'TRACE\_TRACELEVEL'\n
+     overwrites the value of 'TRACE\_TRACELEVEL'\n
+\li 'TRACESESSION\_' + sessionId + '\_DIRECTORY'\n
+     same as 'TRACE\_DIRECTORY'\n
+     overwrites the value of 'TRACE\_DIRECTORY'\n
+\li Configuration values, overwrite the setting from the environment:
+\li 'TRACES.TrcServ.TraceDirectory'\n
+     The trace files are stored in this directory; relative pathes apply to the directory of the executable\n
+     default: 'trc'\n
+\li 'TRACES.' + sessionId + '.TraceLevel'\n
+     '1'-'9': only trace with a tracelevel lower or equal than the this value are written\n
+     default: '5'\n
+\li 'TRACES.' + sessionId + '.Topic\_' + topic\n
+     'inactive': do not write traces for this topic\n
+     'active':   write traces for this topic\n
+     default: 'true'\n
+\li Tracelevel:\n
+   1, 'MajorError',        'Serious Errors - highest priority'\n
+   2, 'MinorError',        'Less serious Errors'\n
+   3, 'MajorWarning',      'General warnings of higher priority'\n
+   4, 'MinorWarning',      'General warnings of lower priority'\n
+   5, 'BasicRuntime',      'Basic runtime information'\n
+   6, 'BasicProcess',      'Including process data'\n
+   7, 'SecondaryRuntime',  'Secondary runtime information'\n
+   8, 'SecondaryFrequent', 'Including polling of events'\n
+   9, 'All',               'All'\n
 
-/*! \page trace trace library - Release Notes
- *
- *  @version 1.0.0.5
- *
- *  \section change1 Change history
- *
- *  \date 23.10.2013 \li V1.0.0.5
- *                   \li bugfix -  Mantis 0029551: Buildsystem: Integrate Boost 1.54
- *
- *  \date 19.10.2012 \li V1.0.0.4
- *                   \li bugfix -  Mantis 0024741: TraceDLL: use current working directory for relative paths
- *                   \li bugfix -  Mantis 0024740: TraceDLL: add Interface to request TraceLevel/Topic
- *                  new header SelfServiceCommon\Interfaces\inc\trc\TrcHdlLevel.h
- *                             SelfServiceCommon\Interfaces\inc\trc\TraceLevel.h
- *
- *  \date 19.03.2012 \li V1.0.0.3
- *                   \li SteT -  Mantis 0017128: TrcServ: replace communication layer
- *                       conflict with massai-tools.lib renamed TraceCallback to TraceLineCallback
- *
- *  \date 23.02.2012 \li V1.0.0.2
- *                   \li SteT Mantis 0022536: trace.dll: do not add \n to \r; use mode binary
- *
- *  \date 11.01.2012 \li V1.0.0.1
- *                   \li SteT Mantis 0022076: Trace.dll: write traces before openTrace and after closeTrace to stdout
- *
- *  \date 27.10.2011 \li V1.0.0.0
- *                   \li SteT Initial Version
- *
- *   sessionId: replace '<>:'/\|?*.' by '_'
- *   topic:     replace '.' by '_'
- *
- *
- *   Settings read from environment:
- *
- *     'TRACE_USE_CONFIG':
- *       'false'/'inactive': do not use configuration
- *       'true'/'active':    read entries from configuration
- *       default: 'true'
- *
- *     'TRACE_TOPIC_' + topic
- *       'false'/'inactive': do not write traces for this topic
- *       'true'/'active':    write traces for this topic
- *       default: 'true'
- *
- *     'TRACE_TRACELEVEL'
- *       '1'-'9': only trace with a tracelevel lower or equal than the this value are written
- *       default: '5'
- *
- *     'TRACE_DIRECTORY'
- *       The trace files are stored in this directory; relative pathes apply to the directory of the executable
- *       default: ''
- *
- *     'TRACESESSION_' + sessionId + '_TOPIC_' + topic
- *       same as 'TRACE_TOPIC_' + topic
- *       overwrites the value of 'TRACE_TOPIC_' + topic
- *
- *     'TRACESESSION_' + sessionId + '_TRACELEVEL'
- *       same as 'TRACE_TRACELEVEL'
- *       overwrites the value of 'TRACE_TRACELEVEL'
- *
- *     'TRACESESSION_' + sessionId + '_DIRECTORY'
- *       same as 'TRACE_DIRECTORY'
- *       overwrites the value of 'TRACE_DIRECTORY'
- *
- *
- *   Configuration values, overwrite the setting from the environment:
- *
- *     'TRACES.TrcServ.TraceDirectory'
- *       The trace files are stored in this directory; relative pathes apply to the directory of the executable
- *       default: 'trc'
- *
- *     'TRACES.' + sessionId + '.TraceLevel'
- *       '1'-'9': only trace with a tracelevel lower or equal than the this value are written
- *       default: '5'
- *
- *     'TRACES.' + sessionId + '.Topic_' + topic
- *       'inactive': do not write traces for this topic
- *       'active':   write traces for this topic
- *       default: 'true'
- *
- *
- *   Tracelevel:
- *
- *     1, 'MajorError',        'Serious Errors - highest priority'
- *     2, 'MinorError',        'Less serious Errors'
- *     3, 'MajorWarning',      'General warnings of higher priority'
- *     4, 'MinorWarning',      'General warnings of lower priority'
- *     5, 'BasicRuntime',      'Basic runtime information'
- *     6, 'BasicProcess',      'Including process data'
- *     7, 'SecondaryRuntime',  'Secondary runtime information'
- *     8, 'SecondaryFrequent', 'Including polling of events'
- *     9, 'All',               'All'
- *
  */
diff --git a/SelfServiceCommon/Massai/cpp/Trace/res/TraceDLL.rc b/SelfServiceCommon/Massai/cpp/Trace/res/TraceDLL.rc
index 487762c4..85989b4f 100644
--- a/SelfServiceCommon/Massai/cpp/Trace/res/TraceDLL.rc
+++ b/SelfServiceCommon/Massai/cpp/Trace/res/TraceDLL.rc
@@ -35,5 +35,3 @@ BEGIN
         VALUE "Translation", 0x407, 1200
     END
 END
-
-
diff --git a/SelfServiceCommon/Massai/cpp/initIDL/src/local_def.cpp b/SelfServiceCommon/Massai/cpp/initIDL/src/local_def.cpp
index e817a5a1..212148a4 100644
--- a/SelfServiceCommon/Massai/cpp/initIDL/src/local_def.cpp
+++ b/SelfServiceCommon/Massai/cpp/initIDL/src/local_def.cpp
@@ -1,17 +1,24 @@
-#include "versionlib.h"
+#include "tools/versionlib.h"
 
-/*! \page initidl initIDL - Release Notes
- *
- *
- *  \date 23.09.2011 \li V1.0.0.1
- *                   \li JK - Modify "gmrdev.idl" to include dataType
- *                   \li      handling.
- *                   \li    - Remove "gmrdev_bcr.idl", functionality is
- *                   \li      now in "gmrdev.idl".
- *
- *  \date 08.12.2010 \li V1.0.0.0
- *                   \li SteT Initial Version
- *
- */
+VERSION_ID_DT(initIDL, 1, 0, 0, 1);
 
-VERSION_ID_DT(initIDL, 1, 0, 0, 1);
\ No newline at end of file
+/*!
+\page initIDL.lib Release Notes initIDL.lib
+
+@version 1.0.0.1
+
+\section changehistory Change history
+
+initIDL.lib
+
+\version 1.0.0.1
+\li 23.09.2011
+\li JK - Modify "gmrdev.idl" to include dataType
+\li      handling.
+\li    - Remove "gmrdev_bcr.idl", functionality is
+\li      now in "gmrdev.idl".
+
+\version 1.0.0.0
+\li 08.12.2010
+\li SteT Initial Version
+*/
diff --git a/SelfServiceCommon/Massai/cpp/textIDL/src/local_def.cpp b/SelfServiceCommon/Massai/cpp/textIDL/src/local_def.cpp
index 46695995..0b276427 100644
--- a/SelfServiceCommon/Massai/cpp/textIDL/src/local_def.cpp
+++ b/SelfServiceCommon/Massai/cpp/textIDL/src/local_def.cpp
@@ -1,48 +1,60 @@
-#include "versionlib.h"
+#include "tools/versionlib.h"
 
 VERSION_ID_DT(textIDL, 1, 0, 0, 8);
 
-/*! \page textidl textidl - Release Notes
+/*!
+\page textidl.lib Release Notes - textidl.lib
 
-\date 15.08.2013
-\li V1.0.0.8
-\li Mantis 0028544: GPP: Volke/Rickert/H&W-Printer sends initially MEDIA_ABSENT when status is unknown
-added mediaStatusType.MEDIA_UNKNOWN
-
-
-\date 05.07.2017
-\li V1.0.0.7
-applicationStart.cmdLine changed from String to List of Strings
-
-\date 05.07.2017 \li V1.0.0.6
-                 \li SteT Mantis 23800: cussif: SITA platform sends activation event with simple type, cussif throws BadKind
+@version 1.0.0.8
 
-\date 20.12.2011 \li V1.0.0.5
-                 \li SteT Mantis 0022538: Platform_Extension_CUSS_1_2: IDL use cuss 1.2 idls
-                       added:
-                          STATE_EXPLANATION
-                          ACTIVE_TRANSFER
-                          TRANSACTION_EXPLANATION
+\section changehistory Change history
 
-\date 20.12.2011 \li V1.0.0.4
-                 \li SteT textappcontrol added
+textidl.lib
 
-\date 08.11.2011 \li V1.0.0.3
-                 \li SteT cuss codes extended
-                       added:
-                          BAGGAGE_WEIGHT_OUT_OF_RANGE
-
-\date 28.07.2011 \li V1.0.0.2
-                 \li SteT cuss codes extended
-                       added:
-                          BAGGAGE_INVALID_DATA
-                          BAGGAGE_TRANSPORT_FAILED
-
-\date 22.06.2011 \li V1.0.0.1
-                 \li SteT cuss codes added
+\version 1.0.0.8
+\li 15.08.2013
+\li Mantis 0028544: GPP: Volke/Rickert/H&W-Printer sends initially MEDIA_ABSENT when status is unknown\n
+added mediaStatusType.MEDIA_UNKNOWN
 
-\date 08.12.2010 \li V1.0.0.0
-                 \li SteT Initial Version
 
+\version 1.0.0.7
+\li 05.07.2017
+\li applicationStart.cmdLine changed from String to List of Strings
+
+\version 1.0.0.6
+\li 05.07.2017
+\li SteT Mantis 23800: cussif: SITA platform sends activation event with simple type, cussif throws BadKind
+
+\version 1.0.0.5
+\li 20.12.2011
+\li SteT Mantis 0022538: Platform_Extension_CUSS_1_2: IDL use cuss 1.2 idls
+\li added:\n
+     STATE_EXPLANATION\n
+     ACTIVE_TRANSFER\n
+     TRANSACTION_EXPLANATION
+
+\version 1.0.0.4
+\li 20.12.2011
+\li SteT textappcontrol added
+
+\version 1.0.0.3
+\li 08.11.2011
+\li SteT cuss codes extended
+\li added:\n
+   BAGGAGE_WEIGHT_OUT_OF_RANGE
+
+\version 1.0.0.2
+\li 28.07.2011
+\li SteT cuss codes extended
+\li added:\n
+   BAGGAGE_INVALID_DATA\n
+   BAGGAGE_TRANSPORT_FAILED
+
+\version 1.0.0.1
+\li 22.06.2011
+\li SteT cuss codes added
+
+\version 1.0.0.0
+\li 08.12.2010
+\li SteT Initial Version
 */
-
diff --git a/SelfServiceCommon/Services/cpp/TrcServ/inc/NonCryptingTrcServ_def.h b/SelfServiceCommon/Services/cpp/TrcServ/inc/NonCryptingTrcServ_def.h
index b01247d5..e1825744 100644
--- a/SelfServiceCommon/Services/cpp/TrcServ/inc/NonCryptingTrcServ_def.h
+++ b/SelfServiceCommon/Services/cpp/TrcServ/inc/NonCryptingTrcServ_def.h
@@ -11,13 +11,12 @@
 
 #include "TrcServ_def.h"
 
-
 #undef FILEDESCRIPTION
 #undef INTERNALNAME
 #undef ORIGINALFILENAME
 
-#define FILEDESCRIPTION  "FileDescription"  ,"Non Crypting Trace Service\0"
-#define INTERNALNAME     "InternalName"     ,"NonCryptingTrcServ.exe\0"
-#define ORIGINALFILENAME "OriginalFilename" ,"NonCryptingTrcServ.exe\0"
+#define FILEDESCRIPTION  "FileDescription",  "Non Crypting Trace Service"
+#define INTERNALNAME     "InternalName",     "NonCryptingTrcServ.exe"
+#define ORIGINALFILENAME "OriginalFilename", "NonCryptingTrcServ.exe"
 
 
diff --git a/SelfServiceCommon/Services/cpp/TrcServ/inc/decrypt_def.h b/SelfServiceCommon/Services/cpp/TrcServ/inc/decrypt_def.h
index 89b85cd9..40c0697a 100644
--- a/SelfServiceCommon/Services/cpp/TrcServ/inc/decrypt_def.h
+++ b/SelfServiceCommon/Services/cpp/TrcServ/inc/decrypt_def.h
@@ -1,31 +1,24 @@
-/**
- *  @file      decrypt_def.h
- *
- *  @brief     Version constants
- *
- *  @date      19.09.2007
- *
- *  @author    Materna Information & Communications (SteT)
- *
- ************************************************************************/
-
-
-/*! \page decrypt decrypt - Release Notes
- *
- *  @version V 1.0.0.0
- *
- *  \section change1 Change history
- *
- *  \date 19.09.2007 \li V 1.0.0.0 SteT initial Version
- *
- */
-
 #define FILE_VER_BIN     1,0,0,0
-#define FILE_VER_STR     "FileVersion", "1.0.0.0\0"
+#define FILE_VER_STR     "FileVersion", "1.0.0.0"
+
+/*!
+\page decrypt.exe Release Notes - decrypt.exe
+
+\section changehistory Change history
+
+@version 1.0.0.0
+
+decrypt.exe
+
+\version 1.0.0.0
+\li 19.09.2007
+\li SteT initial Version
+
+ */
 
-#define FILEDESCRIPTION  "FileDescription"  ,"decrypt\0"
-#define INTERNALNAME     "InternalName"     ,"decrypt.exe\0"
-#define LEGALTRADEMARKS  "LegalTrademarks"  ,"\0"
-#define ORIGINALFILENAME "OriginalFilename" ,"decrypt.exe\0"
+#define FILEDESCRIPTION  "FileDescription",  "decrypt"
+#define INTERNALNAME     "InternalName",     "decrypt.exe"
+#define ORIGINALFILENAME "OriginalFilename", "decrypt.exe"
+#define LEGALTRADEMARKS  "LegalTrademarks",  ""
 
 
-- 
2.41.0.windows.1

